<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Patterns of nucleotide and amino acid substitution</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <link rel="stylesheet" href="pandoc.css" />
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js" type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Patterns of nucleotide and amino acid substitution</h1>
</header>
<nav id="TOC" role="doc-toc">
<ul>
<li><a href="#introduction">Introduction</a></li>
<li><a href="#the-genetic-code">The genetic code</a></li>
<li><a href="#rates-of-synonymous-and-non-synonymous-substitution">Rates of synonymous and non-synonymous substitution</a></li>
<li><a href="#revising-the-neutral-theory">Revising the neutral theory</a></li>
<li><a href="#creative-commons-license">Creative Commons License</a></li>
</ul>
</nav>
<h1 class="unnumbered" id="introduction">Introduction</h1>
<p>So I’ve just suggested that the neutral theory of molecular evolution explains quite a bit, but it also ignores quite a bit. The derivations we did assumed that all substitutions are equally likely to occur, because they are selectively neutral. That isn’t plausible. We need look no further than sickle cell anemia to see an example of a protein polymorphism in which a single nucleotide substitution and a single amino acid difference has a very large effect on fitness. Even reasoning from first principles we can see that it doesn’t make much sense to think that all nucleotide substitutions are created equal. Just as it’s unlikely that you’ll improve the performance of your car if you pick up a sledgehammer, open its hood, close your eyes, and hit something inside, so it’s unlikely that picking a random amino acid in a protein and substituting it with a different one will improve the function of the protein.<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a></p>
<h1 class="unnumbered" id="the-genetic-code">The genetic code</h1>
<p>Of course, not all nucleotide sequence substitutions lead to amino acid substitutions in protein-coding genes. There is redundancy in the genetic code. Table <a href="#table:code" data-reference-type="ref" data-reference="table:code">1</a> is a list of the codons in the universal genetic code.<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a> Notice that there are only two amino acids, methionine and tryptophan, that have a single codon. All the rest have at least two. Serine, arginine, and leucine have six.</p>
<div id="table:code">
<table>
<caption>The universal genetic code.</caption>
<tbody>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">Amino</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Amino</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Amino</td>
<td style="text-align: left;"></td>
<td style="text-align: left;">Amino</td>
</tr>
<tr class="even">
<td style="text-align: left;">Codon</td>
<td style="text-align: left;">Acid</td>
<td style="text-align: left;">Codon</td>
<td style="text-align: left;">Acid</td>
<td style="text-align: left;">Codon</td>
<td style="text-align: left;">Acid</td>
<td style="text-align: left;">Codon</td>
<td style="text-align: left;">Acid</td>
</tr>
<tr class="odd">
<td style="text-align: left;">UUU</td>
<td style="text-align: left;">Phe</td>
<td style="text-align: left;">UCU</td>
<td style="text-align: left;">Ser</td>
<td style="text-align: left;">UAU</td>
<td style="text-align: left;">Tyr</td>
<td style="text-align: left;">UGU</td>
<td style="text-align: left;">Cys</td>
</tr>
<tr class="even">
<td style="text-align: left;">UUC</td>
<td style="text-align: left;">Phe</td>
<td style="text-align: left;">UCC</td>
<td style="text-align: left;">Ser</td>
<td style="text-align: left;">UAC</td>
<td style="text-align: left;">Tyr</td>
<td style="text-align: left;">UGC</td>
<td style="text-align: left;">Cys</td>
</tr>
<tr class="odd">
<td style="text-align: left;">UUA</td>
<td style="text-align: left;">Leu</td>
<td style="text-align: left;">UCA</td>
<td style="text-align: left;">Ser</td>
<td style="text-align: left;">UAA</td>
<td style="text-align: left;">Stop</td>
<td style="text-align: left;">UGA</td>
<td style="text-align: left;">Stop</td>
</tr>
<tr class="even">
<td style="text-align: left;">UUG</td>
<td style="text-align: left;">Leu</td>
<td style="text-align: left;">UCG</td>
<td style="text-align: left;">Ser</td>
<td style="text-align: left;">UAG</td>
<td style="text-align: left;">Stop</td>
<td style="text-align: left;">UGG</td>
<td style="text-align: left;">Trp</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">CUU</td>
<td style="text-align: left;">Leu</td>
<td style="text-align: left;">CCU</td>
<td style="text-align: left;">Pro</td>
<td style="text-align: left;">CAU</td>
<td style="text-align: left;">His</td>
<td style="text-align: left;">CGU</td>
<td style="text-align: left;">Arg</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CUC</td>
<td style="text-align: left;">Leu</td>
<td style="text-align: left;">CCC</td>
<td style="text-align: left;">Pro</td>
<td style="text-align: left;">CAC</td>
<td style="text-align: left;">His</td>
<td style="text-align: left;">CGC</td>
<td style="text-align: left;">Arg</td>
</tr>
<tr class="even">
<td style="text-align: left;">CUA</td>
<td style="text-align: left;">Leu</td>
<td style="text-align: left;">CCA</td>
<td style="text-align: left;">Pro</td>
<td style="text-align: left;">CAA</td>
<td style="text-align: left;">Gln</td>
<td style="text-align: left;">CGA</td>
<td style="text-align: left;">Arg</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CUG</td>
<td style="text-align: left;">Leu</td>
<td style="text-align: left;">CCG</td>
<td style="text-align: left;">Pro</td>
<td style="text-align: left;">CAG</td>
<td style="text-align: left;">Gln</td>
<td style="text-align: left;">CGG</td>
<td style="text-align: left;">Arg</td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">AUU</td>
<td style="text-align: left;">Ile</td>
<td style="text-align: left;">ACU</td>
<td style="text-align: left;">Thr</td>
<td style="text-align: left;">AAU</td>
<td style="text-align: left;">Asn</td>
<td style="text-align: left;">AGU</td>
<td style="text-align: left;">Ser</td>
</tr>
<tr class="even">
<td style="text-align: left;">AUC</td>
<td style="text-align: left;">Ile</td>
<td style="text-align: left;">ACC</td>
<td style="text-align: left;">Thr</td>
<td style="text-align: left;">AAC</td>
<td style="text-align: left;">Asn</td>
<td style="text-align: left;">AGC</td>
<td style="text-align: left;">Ser</td>
</tr>
<tr class="odd">
<td style="text-align: left;">AUA</td>
<td style="text-align: left;">Ile</td>
<td style="text-align: left;">ACA</td>
<td style="text-align: left;">Thr</td>
<td style="text-align: left;">AAA</td>
<td style="text-align: left;">Lys</td>
<td style="text-align: left;">AGA</td>
<td style="text-align: left;">Arg</td>
</tr>
<tr class="even">
<td style="text-align: left;">AUG</td>
<td style="text-align: left;">Met</td>
<td style="text-align: left;">ACG</td>
<td style="text-align: left;">Thr</td>
<td style="text-align: left;">AAG</td>
<td style="text-align: left;">Lys</td>
<td style="text-align: left;">AGG</td>
<td style="text-align: left;">Arg</td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">GUU</td>
<td style="text-align: left;">Val</td>
<td style="text-align: left;">GCU</td>
<td style="text-align: left;">Ala</td>
<td style="text-align: left;">GAU</td>
<td style="text-align: left;">Asp</td>
<td style="text-align: left;">GGU</td>
<td style="text-align: left;">Gly</td>
</tr>
<tr class="odd">
<td style="text-align: left;">GUC</td>
<td style="text-align: left;">Val</td>
<td style="text-align: left;">GCC</td>
<td style="text-align: left;">Ala</td>
<td style="text-align: left;">GAC</td>
<td style="text-align: left;">Asp</td>
<td style="text-align: left;">GGC</td>
<td style="text-align: left;">Gly</td>
</tr>
<tr class="even">
<td style="text-align: left;">GUA</td>
<td style="text-align: left;">Val</td>
<td style="text-align: left;">GCA</td>
<td style="text-align: left;">Ala</td>
<td style="text-align: left;">GAA</td>
<td style="text-align: left;">Glu</td>
<td style="text-align: left;">GGA</td>
<td style="text-align: left;">Gly</td>
</tr>
<tr class="odd">
<td style="text-align: left;">GUG</td>
<td style="text-align: left;">Val</td>
<td style="text-align: left;">GCG</td>
<td style="text-align: left;">Ala</td>
<td style="text-align: left;">GAG</td>
<td style="text-align: left;">Glu</td>
<td style="text-align: left;">GGG</td>
<td style="text-align: left;">Gly</td>
</tr>
</tbody>
</table>
</div>
<p>Moreover, most of the redundancy is in the third position, where we can distinguish 2-fold from 4-fold redundant sites (Table <a href="#table:fold" data-reference-type="ref" data-reference="table:fold">2</a>). 2-fold redundant sites are those at which either one of two nucleotides can be present in a codon for a single amino acid. 4-fold redundant sites are those at which any of the four nucleotides can be present in a codon for a single amino acid. In some cases there is redundancy in the first codon position, e.g, both AGA and CGA are codons for arginine. Thus, many nucleotide substitutions at third positions do not lead to amino acid substitutions, and some nucleotide substitutions at first positions do not lead to amino acid substitutions. But every nucleotide substitution at a second codon position leads to an amino acid substitution. Nucleotide substitutions that do not lead to amino acid substitutions are referred to as <span><em>synonymous substitutions</em></span>, because the codons involved are synonymous, i.e., code for the same amino acid. Nucleotide substitutions that do lead to amino acid substituions are <span><em>non-synonymous substitutions</em></span>.</p>
<div id="table:fold">
<table>
<caption>Examples of 4-fold and 2-fold redundancy in the 3rd position of the universal genetic code.</caption>
<tbody>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: left;">Amino</td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">Codon</td>
<td style="text-align: left;">Acid</td>
<td style="text-align: left;">Redundancy</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CCU</td>
<td style="text-align: left;">Pro</td>
<td style="text-align: left;">4-fold</td>
</tr>
<tr class="even">
<td style="text-align: left;">CCC</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">CCA</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">CCG</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">AAU</td>
<td style="text-align: left;">Asn</td>
<td style="text-align: left;">2-fold</td>
</tr>
<tr class="even">
<td style="text-align: left;">AAC</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">AAA</td>
<td style="text-align: left;">Lys</td>
<td style="text-align: left;">2-fold</td>
</tr>
<tr class="even">
<td style="text-align: left;">AAG</td>
<td style="text-align: left;"></td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</div>
<h1 class="unnumbered" id="rates-of-synonymous-and-non-synonymous-substitution">Rates of synonymous and non-synonymous substitution</h1>
<p>By using a modification of the simple Jukes-Cantor model we encountered before, it is possible to make separate estimates of the number of synonymous substitutions and of the number of non-synonymous substitutions that have occurred since two sequences diverged from a common ancestor. If we combine an estimate of the <span><em>number</em></span> of differences with an estimate of the <span><em>time of divergence</em></span> we can estimate the rates of synonymous and non-synonymous substitution (number/time). Table <a href="#table:substitution-data" data-reference-type="ref" data-reference="table:substitution-data">3</a> shows some representative estimates for the rates of synonymous and non-synonymous substitution in different genes studied in mammals.</p>
<div id="table:substitution-data">
<table>
<caption>Representative rates of synonymous and non-synonymous substitution in mammalian genes (from <span class="citation" data-cites="Li97"></span>). Rates are expressed as the number of substitutions per <span class="math inline">\(10^9\)</span> years.</caption>
<thead>
<tr class="header">
<th style="text-align: left;">Locus</th>
<th style="text-align: center;">Non-synonymous rate</th>
<th style="text-align: center;">Synonymous rate</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Histone</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">H4</td>
<td style="text-align: center;">0.00</td>
<td style="text-align: center;">3.94</td>
</tr>
<tr class="odd">
<td style="text-align: left;">H2</td>
<td style="text-align: center;">0.00</td>
<td style="text-align: center;">4.52</td>
</tr>
<tr class="even">
<td style="text-align: left;">Ribosomal proteins</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">S17</td>
<td style="text-align: center;">0.06</td>
<td style="text-align: center;">2.69</td>
</tr>
<tr class="even">
<td style="text-align: left;">S14</td>
<td style="text-align: center;">0.02</td>
<td style="text-align: center;">2.16</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Hemoglobins &amp; myoglobin</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\alpha\)</span>-globin</td>
<td style="text-align: center;">0.56</td>
<td style="text-align: center;">4.38</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(\beta\)</span>-globin</td>
<td style="text-align: center;">0.78</td>
<td style="text-align: center;">2.58</td>
</tr>
<tr class="even">
<td style="text-align: left;">Myoglobin</td>
<td style="text-align: center;">0.57</td>
<td style="text-align: center;">4.10</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Interferons</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\gamma\)</span></td>
<td style="text-align: center;">3.06</td>
<td style="text-align: center;">5.50</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(\alpha\)</span>1</td>
<td style="text-align: center;">1.47</td>
<td style="text-align: center;">3.24</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\beta\)</span>1</td>
<td style="text-align: center;">2.38</td>
<td style="text-align: center;">5.33</td>
</tr>
</tbody>
</table>
</div>
<p>Two very important observations emerge after you’ve looked at this table for awhile. The first won’t come as any shock. The rate of non-synonymous substitution is generally lower than the rate of synonymous substitution. This is a result of the “sledgehammer principle” I mentioned earlier. Just as taking a sledgehammer to your car engine and making random changes is unlikely to make it run better, so making random changes to the amino acid composition of a protein is unlikely to make it function better. Mutations that change the amino acid sequence of a protein are more likely to reduce that protein’s functionality than to increase it. As a result, they are likely to lower the fitness of individuals carrying them, and they will have a lower probability of being fixed than those mutations that do not change the amino acid sequence.<a href="#fn3" class="footnote-ref" id="fnref3" role="doc-noteref"><sup>3</sup></a></p>
<p>The second observation is more subtle. Rates of non-synonymous substitution vary by more than two orders of magnitude: 0.02 substitutions per nucleotide per billion years in ribosomal protein S14 to 3.06 substitutions per nucleotide per billion years in <span class="math inline">\(\gamma\)</span>-interferon, while rates of synonymous substitution vary only by a factor of two (2.16 in ribosomal protein S14 to 5.50 in <span class="math inline">\(\gamma\)</span> interferons. If synonymous substitutions are neutral, as they probably are to a first approximation,<a href="#fn4" class="footnote-ref" id="fnref4" role="doc-noteref"><sup>4</sup></a> then the rate of synonymous substitution should equal the mutation rate. Thus, the rate of synonymous substitution should be approximately the same at every locus, which is roughly what we observe. But proteins differ in the degree to which their physiological function affects the performance and fitness of the organisms that carry them. Some, like histones and ribosomal proteins, are intimately involved with chromatin or translation of messenger RNA into protein. It’s easy to imagine that just about any change in the amino acid sequence of such proteins will have a detrimental effect on their function. Others, like interferons, are involved in responses to viral or bacterial pathogens. It’s easy to imagine not only that the selection on these proteins might be less intense, but that some amino acid substitutions might actually be favored by natural selection because they enhance resistance to certain strains of pathogens. Thus, the probability that a non-synonymous substitution will be fixed is likely to vary substantially among genes, just as we observe.</p>
<h1 class="unnumbered" id="revising-the-neutral-theory">Revising the neutral theory</h1>
<p>So we’ve now produced empirical evidence that many mutations are <span> <em>not</em></span> neutral. Does this mean that we throw the neutral theory of molecular evolution away? Hardly. We need only modify it a little to accomodate these new observations.</p>
<ul>
<li><p><span><em>Most non-synonymous substitutions are deleterious.</em></span> We can actually generalize this assertion a bit and say that most mutations that affect function are deleterious. After all, organisms have been evolving for about 3.5 billion years. Wouldn’t you expect their cellular machinery to work pretty well by now?</p></li>
<li><p><span><em>Most molecular variability found in natural populations is selectively neutral.</em></span> If most function-altering mutations are deleterious, it follows that we are unlikely to find much variation in populations for such mutations. Selection will quickly eliminate them.<a href="#fn5" class="footnote-ref" id="fnref5" role="doc-noteref"><sup>5</sup></a></p></li>
<li><p><span><em>Natural selection is primarily purifying.</em></span> Although natural selection for variants that improve function is ultimately the source of adaptation, even at the molecular level, most of the time selection is simply eliminating variants that are less fit than the norm, not promoting the fixation of new variants that increase fitness.</p></li>
<li><p><span><em>Alleles enhancing fitness are rapidly incorporated.</em></span><a href="#fn6" class="footnote-ref" id="fnref6" role="doc-noteref"><sup>6</sup></a> They do not remain polymorphic for long, so we aren’t likely to find them when they’re polymorphic.</p></li>
</ul>
<p>As we’ll see, even these revisions aren’t entirely sufficient, but what we do from here on out is more to provide refinements and clarifications than to undertake wholesale revisions.</p>
<h1 class="unnumbered" id="creative-commons-license">Creative Commons License</h1>
<p>These notes are licensed under the Creative Commons Attribution License. To view a copy of this license, visit https://creativecommons.org/licenses/by/4.0/ or send a letter to Creative Commons, 559 Nathan Abbott Way, Stanford, California 94305, USA.</p>
<section class="footnotes" role="doc-endnotes">
<hr />
<ol>
<li id="fn1" role="doc-endnote"><p>Obviously it happens sometimes. If it didn’t, there wouldn’t be any adaptive evolution. It’s just that, on average, mutations are more likely to decrease fitness than to increase it.<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2" role="doc-endnote"><p>By the way, the “universal” genetic code is not universal. There are at least 31 (<a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi">https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi</a>), but all of them have similar redundancy properties.<a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn3" role="doc-endnote"><p>Remember our discussion of the probability that disfavored mutations are fixed as a result of natural selection. They can be fixed, but they are less likely to be fixed than those that are neutral.<a href="#fnref3" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn4" role="doc-endnote"><p>We’ll see that they may not be completely neutral a little later, but at least it’s reasonable to believe that the intensity of selection to which they are subject is a lot less than that to which non-synonymous substitutions are subject.<a href="#fnref4" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn5" role="doc-endnote"><p>Remember, when I say that “the variability is selectively neutral,” that’s shorthand for saying that “the product of effective population size and the selection coefficient on different alleles is less than one, meaning that the dynamics of allele frequency change are more similar to those of an allele that has no effects on fitness than to those of an allele with an effect on fitness when we can neglect genetic drift.”<a href="#fnref5" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn6" role="doc-endnote"><p>To be more precise I should have written <span><em>Alleles enhancing fitness are rapidly incorporated, <span><strong>when they are not lost quickly as a result of genetic drift</strong></span>.</em></span><a href="#fnref6" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section>
</body>
</html>
