<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Introduction to molecular population genetics</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    /* The extra [class] is a hack that increases specificity enough to
       override a similar rule in reveal.js */
    ul.task-list[class]{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      font-size: inherit;
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
  </style>
  <link rel="stylesheet" href="pandoc.css" />
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script
  src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js"
  type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Introduction to molecular population genetics</h1>
</header>
<nav id="TOC" role="doc-toc">
<ul>
<li><a href="#introduction" id="toc-introduction">Introduction</a></li>
<li><a href="#types-of-data" id="toc-types-of-data">Types of data</a>
<ul>
<li><a href="#the-physical-basis-of-molecular-variation"
id="toc-the-physical-basis-of-molecular-variation">The physical basis of
molecular variation</a></li>
<li><a href="#revealing-molecular-variation"
id="toc-revealing-molecular-variation">Revealing molecular
variation</a></li>
</ul></li>
<li><a href="#divergence-of-nucleotide-sequences"
id="toc-divergence-of-nucleotide-sequences">Divergence of nucleotide
sequences</a></li>
<li><a href="#the-neutral-theory-of-molecular-evolution"
id="toc-the-neutral-theory-of-molecular-evolution">The neutral theory of
molecular evolution</a></li>
<li><a href="#early-empirical-observations"
id="toc-early-empirical-observations">Early empirical
observations</a></li>
<li><a href="#neutral-substitutions-and-neutral-variation"
id="toc-neutral-substitutions-and-neutral-variation">Neutral
substitutions and neutral variation</a>
<ul>
<li><a href="#the-rate-of-molecular-evolution"
id="toc-the-rate-of-molecular-evolution">The rate of molecular
evolution</a></li>
<li><a href="#diversity-in-populations"
id="toc-diversity-in-populations">Diversity in populations</a></li>
</ul></li>
<li><a href="#conclusions" id="toc-conclusions">Conclusions</a></li>
<li><a href="#creative-commons-license"
id="toc-creative-commons-license">Creative Commons License</a></li>
</ul>
</nav>
<h1 class="unnumbered" id="introduction">Introduction</h1>
<p>The study of evolutionary biology is commonly divided into two
components: study of the <span><em>processes</em></span> by which
evolutionary change occurs and study of the
<span><em>patterns</em></span> produced by those processes. By “pattern”
we mean primarily the pattern of phylogenetic relationships among
species or genes.<a href="#fn1" class="footnote-ref" id="fnref1"
role="doc-noteref"><sup>1</sup></a> Studies of evolutionary processes
often don’t often devote too much attention to evolutionary patterns,
except insofar as it is often necessary to take account of evolutionary
history in determining whether or not a particular feature is an
adaptation. Similarly, studies of evolutionary pattern sometimes try not
to use any knowledge of evolutionary processes to improve their guesses
about phylogenetic relationships, because the relationship between
process and pattern can be tenuous.<a href="#fn2" class="footnote-ref"
id="fnref2" role="doc-noteref"><sup>2</sup></a> Those who take this
approach argue that invoking a particular evolutionary process seems
often to be a way of making sure that you get the pattern you want to
get from the data.</p>
<p>Or at least that’s the way it was in evolutionary biology when
evolutionary biologists were concerned primarily with the evolution of
morphological, behavioral, and physiological traits and when
systematists used primarily anatomical, morphological, and chemical
features (but not proteins or DNA) to describe evolutionary patterns.
With the advent of molecular biology after the Second World War and its
application to an increasing diversity of organisms in the late 1950s
and early 1960s, that began to change. Goodman <span class="citation"
data-cites="Goodman62"></span> used the degree of immunological
cross-reactivity between serum proteins as an indication of the
evolutionary distance among primates. Zuckerkandl and Pauling <span
class="citation" data-cites="Zuckerkandl-Pauling65"></span> proposed
that after species diverged, their proteins diverged according to a
“molecular clock,” suggesting that molecular similarities could be used
to reconstruct evolutionary history. In 1966, Harris <span
class="citation" data-cites="Harris66"></span> and Lewontin and
Hubby <span class="citation"
data-cites="Hubby-Lewontin66 Lewontin-Hubby66"></span> showed that human
populations and populations of <span><em>Drosophila
pseudoobscura</em></span> respectively, contained surprising amounts of
genetic diversity.</p>
<p>We’ll focus first on advances made in understanding the processes of
molecular evolution. Once we have a passing understanding of those
processes, we’ll shift to topics that are generally more interesting to
evolutionary biologists, i.e., making inferences about evolutionary
patterns from molecular data. Up to this point in the course we’ve
completely ignored evolutionary pattern.<a href="#fn3"
class="footnote-ref" id="fnref3" role="doc-noteref"><sup>3</sup></a> As
you’ll see in what follows, however, any discussion of molecular
evolution, even if it focuses on understanding the processes, cannot
avoid some careful attention to the pattern.</p>
<h1 class="unnumbered" id="types-of-data">Types of data</h1>
<p>If you’re interested in the history of molecular evolution, you may
be interested in this review of the types of data that population
geneticists have used in the last 50 or 60 years to provide insights
into evolutionary processes. If you’re not interested in the history,
feel free to skip this section. I will touch on only a couple of the
real high points during lecture. Much of the data being collected now
for population genetics is treated as single nucleotide
polymorphisms (sometimes with genetic linkage taken into account) or
copy-number variation, and it is derived either from low-coverage
whole-genome resequencing or from a reduced representation sequencing
approach like some version of RADseq or genotyping-by-sequencing. The
exceptions are that for some purposes, microsatellites are still the
marker of choice and for others, RNAseq can be used to explore
differences in gene expression between individuals or under different
conditions.</p>
<p>We’ve already encountered a couple of these (microsatellites and
SNPs), but there are a variety of important categories into which we can
group data used for molecular evolutionary analyses. Even though studies
of molecular evolution in the last 20-25 years have focused mostly on
data derived from DNA sequence or copy number variation, modern
applications of molecular data evolved from earlier applications.
Markers that were used before the advent of (relatively) easy and cheap
DNA sequencing had their limitations, but analyses of those data also
laid the groundwork for most or all of what’s going on in analyses of
molecular evolution today. Thus, it’s useful to remind everyone what
kinds of molecular data have been used to provide insight into
evolutionary patterns and processes and to agree on some terminology for
the ones we’ll say something about. Let’s talk first about the physical
basis of the underlying data. Then we’ll talk about the laboratory
methods used to reveal variation.</p>
<h2 class="unnumbered"
id="the-physical-basis-of-molecular-variation">The physical basis of
molecular variation</h2>
<p>With the exception of RNA viruses, the hereditary information in all
organisms is carried in DNA. Ultimately, differences in any of the
molecular markers we study (and of genetically-based morphological,
behavioral, or physiological traits) is associated with some difference
in the physical structure of DNA.</p>
<dl>
<dt>Nucleotide sequence</dt>
<dd>
<p>A difference in nucleotide sequence is the most obvious way in which
two homologous stretches of DNA may differ. The differences may be in
translated portions of protein genes (exons), portions of protein genes
that are transcribed but not translated (e.g., introns, 5’ or 3’
untranslated regions), non-transcribed functional regions (e.g.,
promoters), or regions without apparent function.</p>
</dd>
<dt>Protein sequence</dt>
<dd>
<p>Because of redundancy in the genetic code, a difference in nucleotide
sequence at a protein-coding locus may or may not result in proteins
with a different amino acid sequence. <span><strong>Important
note</strong></span>: Don’t forget that some loci code for RNA that has
an immediate function without being translated to a protein, e.g.,
ribosomal RNA and various small nuclear RNAs.</p>
</dd>
<dt>Secondary, tertiary, and quaternary structure</dt>
<dd>
<p>Differences in amino acid sequence may or may not lead to a different
distribution of <span class="math inline">\(\alpha\)</span>-helices and
<span class="math inline">\(\beta\)</span>-sheets, to a different
three-dimensional structure, or to different multisubunit
combinations.</p>
</dd>
<dt>Imprinting</dt>
<dd>
<p>At certain loci in some organisms the expression pattern of a
particular allele depends on whether that allele was inherited from the
individual’s father or its mother.</p>
</dd>
<dt>Expression</dt>
<dd>
<p>Functional differences among individuals may arise because of
differences in the patterns of gene expression, even if there are no
differences in the primary sequences of the genes that are expressed.<a
href="#fn4" class="footnote-ref" id="fnref4"
role="doc-noteref"><sup>4</sup></a></p>
</dd>
<dt>Sequence organization</dt>
<dd>
<p>Particular genes may differ between organisms because of differences
in the position and number of introns. At the whole genome level, there
may be differences in the amount and kind of repetitive sequences, in
the amount and type of sequences derived from transposable elements, in
the relative proportion of G-C relative to A-T, or even in the identity
and arrangement of genes that are present. In microbial species, only a
subset of genes are present in all strains. For example, in <span>
<em>Streptococcus pneumoniae</em></span> the “core genome” contains only
73% of the loci present in one fully sequenced reference strain <span
class="citation" data-cites="Obert-etal-2006"></span>. Similarly, a
survey of 20 strains of <span><em>Escherichia coli</em></span> and one
of <span><em>E. fergusonii</em></span>, <span> <em>E. coli</em></span>’s
closest relative, identified only 2000 homologous loci that were present
in all strains out of 18,000 orthologous loci identified <span
class="citation" data-cites="Touchon-etal-2009"></span></p>
</dd>
<dt>Copy number variation</dt>
<dd>
<p>Even within diploid genomes, there may be substantial differences in
the number of copies of particular genes. In humans, for example, 76
copy-number polymorphisms (CNPs) were identified in a sample of only 20
individuals, and individuals differed from one another by an average of
11 CNPs. <span class="citation"
data-cites="Sebat-etal-2004"></span>.</p>
</dd>
</dl>
<p>It is worth remembering that in nearly all eukaryotes there are two
different genomes whose characteristics may be analyzed: the nuclear
genome and the mitochondrial genome. In plants there is a third: the
chloroplast genome. In some protists, there may be even more, because of
secondary or tertiary endosymbiosis. The mitochondrial and chloroplast
genomes are typically inherited only through the maternal line, although
some instances of biparental inheritance are known.<a href="#fn5"
class="footnote-ref" id="fnref5" role="doc-noteref"><sup>5</sup></a> In
conifers, chloroplasts are paternally inherited, i.e., through the
pollen parent, and mitochondria are maternally inherited, i.e., through
the seed parent <span class="citation"
data-cites="Neale-Sederoff-1988"></span></p>
<h2 class="unnumbered" id="revealing-molecular-variation">Revealing
molecular variation</h2>
<p>The diversity of laboratory techniques used to reveal molecular
variation is even greater than the diversity of underlying physical
structures. Various techniques involving direct measurement of aspects
of DNA sequence variation are by far the most common today, so I’ll
mention only a few of the techniques that were most widely used in the
past.<a href="#fn6" class="footnote-ref" id="fnref6"
role="doc-noteref"><sup>6</sup></a></p>
<dl>
<dt>Immunological distance</dt>
<dd>
<p>Some molecules, notably protein molecules, induce an immune response
in common laboratory mammals. The extent of cross-reactivity between an
antigen raised to humans and chimps, for example, can be used as a
measure of evolutionary distance. The immunological distance between
humans and chimps is smaller than it is between humans and orangutans,
suggesting that humans and chimps share a more recent common
ancestor.</p>
</dd>
<dt>DNA-DNA hybridization</dt>
<dd>
<p>Once repetitive sequences of DNA have been “subtracted out”,<a
href="#fn7" class="footnote-ref" id="fnref7"
role="doc-noteref"><sup>7</sup></a> the rate and temperature at which
DNA species from two different species anneal reflects the average
percent sequence divergence between them. The percent sequence
divergence can be used as a measure of evolutionary distance.
Immunological distances and DNA-DNA hybridization were once widely used
to identify phylogenetic relationships among species. Neither is now
widely used in molecular evolution studies.</p>
</dd>
<dt>Isozymes</dt>
<dd>
<p>Biochemists recognized in the late 1950s that many soluble enzymes
occurred in multiple forms within a single individual. Population
geneticists, notably Hubby and Lewontin, later recognized that in many
cases, these different forms corresponded to different alleles at a
single locus, <span><em>allozymes</em></span>. Allozymes are relatively
easy to score in most macroscopic organisms, they are typically
co-dominant (the allelic composition of heterozygotes can be inferred),
and they allow investigators to identify both variable and non-variable
loci.<a href="#fn8" class="footnote-ref" id="fnref8"
role="doc-noteref"><sup>8</sup></a> Patterns of variation at allozyme
loci may not be representative of genetic variation that does not result
from differences in protein structure or that are related to variation
in proteins that are insoluble.</p>
</dd>
<dt>RFLPs</dt>
<dd>
<p>In the 1970s molecular geneticists discovered restriction enzymes,
enzymes that cleave DNA at specific 4, 5, or 6 base pair sequences, the
<span><em>recognition site</em></span>. A single nucleotide change in a
recognition site is usually enough to eliminate it. Thus, presence or
absence of a restriction site at a particular position in a genome
provides compelling evidence of an underlying difference in nucleotide
sequence at that positon.</p>
</dd>
<dt>RAPDs, AFLPs, ISSRs</dt>
<dd>
<p>With the advent of the polymerase chain reaction in the late 1980s,
several related techniques were developed for the rapid assessment of
genetic variation in organisms for which little or no prior genetic
information was available. These methods differ in details of how the
laboratory procedures are performed, but they are similar in that they
(a) use PCR to amplify anonymous stretches of DNA, (b) generally produce
larger amounts of variation than allozyme analyses of the same taxa, and
(c) are bi-allelic, dominant markers. They have the advantage, relative
to allozymes, that they sample more or less randomly through the genome.
They have the disadvantage that heterozygotes cannot be distinguished
from dominant homozygotes, meaning that it is difficult to use them to
obtain information about levels of within population inbreeding.<a
href="#fn9" class="footnote-ref" id="fnref9"
role="doc-noteref"><sup>9</sup></a></p>
</dd>
<dt>Microsatellites</dt>
<dd>
<p>Satellite DNA, highly repetitive DNA associated with heterochromatin,
had been known since biochemists first began to characterize the
large-scale structure of genomes in DNA-DNA hybridization studies. In
the mid-late 1980s several investigators identified smaller repetitive
units dispersed throughout many genomes. Microsatellites, which consist
of short (2-6) nucleotide sequences repeated many times, have proven
particularly useful for analyses of variation within populations since
the mid-1990s.<a href="#fn10" class="footnote-ref" id="fnref10"
role="doc-noteref"><sup>10</sup></a> Because of high mutation rates at
each locus, they commonly have many alleles. Moreover, they are
typically co-dominant, making them more generally useful than dominant
markers. Identifying variable microsatellite loci is, however, more
laborious than identifying AFLPs, RAPDs, or ISSRs.</p>
</dd>
<dt>Nucleotide sequence</dt>
<dd>
<p>The advent of automated sequencing<a href="#fn11"
class="footnote-ref" id="fnref11" role="doc-noteref"><sup>11</sup></a>
has greatly increased the amount of population-level data available on
nucleotide sequences. The even more recent arrival of high-throughput
DNA sequencing means that sequence information is accumulating even more
rapidly. Nucleotide sequence data has an important advantage over most
of the types of data discussed so far: allozymes, RFLPs, AFLPs, RAPDs,
and ISSRs all hide some amount of nucleotide sequence variation.
Nucleotide sequence differences need not be reflected in any of those
markers. On the other hand, each of those markers provides information
on variation at several or many, independently inherited loci.
Nucleotide sequence information reveals differences at a location that
rarely extends more than 2-3kb. Of course, as next generation sequencing
techniques become less expensive and more widely available, we will see
more and more examples of nucleotide sequence variation from many loci
within individuals.<a href="#fn12" class="footnote-ref" id="fnref12"
role="doc-noteref"><sup>12</sup></a></p>
</dd>
<dt>Single nucleotide polymorphisms</dt>
<dd>
<p>In organisms that are genetically well-characterized, it is possible
to identify a large number of single nucleotide positions that harbor
polymorphisms. SNPs potentially provide high-resolution insight into
patterns of variation within the genome. For example, the HapMap project
identified approximately 3.2M SNPs in the human genome, or about one
every kb <span class="citation" data-cites="HapMap-2007"></span>. With
the advent of RAD-seq, GBS, and similar approaches, it has become
possible to identify large numbers of SNPs even in organisms that are
not genetically well-characterized <span class="citation"
data-cites="Elshire-etal-2011 McKinney-etal-2017"></span>.</p>
</dd>
</dl>
<p>As you can see from these brief descriptions, each of the markers
reveals different aspects of underlying hereditary differences among
individuals, populations, or species. There is no single “best” marker
for evolutionary analyses. Which is best depends on the question you are
asking. In many cases in molecular evolution, the interest is
intrinsically in the evolution of the molecule itself, so the choice is
based not on what those molecules reveal about the organism that
contains them but on what questions about which molecules are the most
interesting.</p>
<h1 class="unnumbered"
id="divergence-of-nucleotide-sequences">Divergence of nucleotide
sequences</h1>
<p>Underlying much of what we’re going to discuss in this part of the
course is the idea that we should be able to describe the degree of
difference between nucleotide sequences, proteins, or anything else as a
result of some underlying evolutionary processes. To illustrate the
principle, let’s start with nucleotide sequences and develop a fairly
simple model that describes how they become different over time.<a
href="#fn13" class="footnote-ref" id="fnref13"
role="doc-noteref"><sup>13</sup></a></p>
<p>Let <span class="math inline">\(q_t\)</span> be the probability that
two homologous nucleotides are identical after having been evolving for
<span class="math inline">\(t\)</span> generations independently since
the gene in which they were found was replicated in their common
ancestor. Let <span class="math inline">\(\lambda\)</span> be the
probability of a substitution<a href="#fn14" class="footnote-ref"
id="fnref14" role="doc-noteref"><sup>14</sup></a> occuring at this
nucleotide position in either of the two genes during a small time
interval, <span class="math inline">\(\Delta t\)</span>. Then <span
class="math display">\[\begin{aligned}
q_{t+\Delta t} &amp;=&amp; (1 - \lambda\Delta t)^2q_t
                  + 2\left(1 - \lambda\Delta t\right)\left({1 \over
                           3}\lambda\Delta t\right)(1 - q_t)
                  + \hbox{o}(\Delta t^2) \\
               &amp;=&amp; (1 - 2\lambda\Delta t)q_t + \left({2 \over
3}\lambda
                                                    \Delta t\right)
                                              (1 - q_t)
                  + \hbox{o}(\Delta t^2) \\
q_{t+\Delta t} - q_t &amp;=&amp; {2 \over 3}\lambda\Delta t - {8 \over
                           3}\lambda\Delta tq_t + \hbox{o}(\Delta t^2)
\\
{q_{t+\Delta t} - q_t \over \Delta t} &amp;=&amp; {2 \over 3}\lambda -
{8 \over
3}\lambda q_t + \hbox{o}(\Delta t) \\
\lim_{\Delta t\to 0}{q_{t+\Delta t} - q_t \over \Delta t}  = {dq_t \over
dt} &amp;=&amp; {2 \over 3}\lambda - {8
\over 3}\lambda q_t \\
    &amp; \mbox{TAMO} &amp; \\
q_t &amp;=&amp; 1 - {3 \over 4}\left(1 - e^{-8\lambda t/3}\right) \\
\end{aligned}\]</span> The expected number of nucleotide substitutions
separating the two sequences at any one position since they diverged is
<span class="math inline">\(d = 2\lambda t\)</span>.<a href="#fn15"
class="footnote-ref" id="fnref15" role="doc-noteref"><sup>15</sup></a>
Thus, <span class="math display">\[\begin{aligned}
  q_t &amp;=&amp; 1 - {3 \over 4}\left(1 - e^{-4d/3}\right) \\
d   &amp;=&amp; -{3 \over 4}\ln\left[1 - {4 \over 3}(1 - q_t)\right] \\
\end{aligned}\]</span> This is the simplest model of nucleotide
substitution possiblethe Jukes-Cantor model <span class="citation"
data-cites="Jukes-Cantor-1969"></span>. It assumes</p>
<ul>
<li><p>that substitutions are equally likely at all positions
and</p></li>
<li><p>that substitution among all nucleotides is equally
likely.</p></li>
</ul>
<p>Let’s examine the second of those assumptions first. Observed
differences between nucleotide sequences shows that some types of
substitutions, i.e., transitions (<span class="math inline">\(A \iff
G\)</span> [purine to purine], <span class="math inline">\(C
\iff T\)</span> [pyrimidine to pyrimidine]), occur much more frequently
than others, i.e., transversions (<span class="math inline">\(A \iff
T\)</span>, <span class="math inline">\(A \iff C\)</span>, <span
class="math inline">\(G \iff C\)</span>, <span class="math inline">\(G
\iff T\)</span> [purine to pyrimidine or vice versa]). There are a
variety of different substitution models corresponding to different
assumed patterns of substitution: Kimura 2 parameter (K2P), Felsenstein
1984 (F84), Hasegawa-Kishino-Yano 1985 (HKY85), Tamura and Nei (TrN),
and generalized time-reversible (GTR). The GTR is, as its name suggests,
the most general <span><em>time-reversible</em></span> model. It allows
substitution rates to differ between each pair of nucleotides. That’s
why it’s general. It still requires, however, that the substitution rate
be the same in both directions. That’s what it means to say that it’s
time reversible. While it is possible to construct a model in which the
substitution rate differs depending on the direction of substitution, it
leads to something of a paradox: with non-reversible substitution models
the distance between two sequences <span
class="math inline">\(A\)</span> and <span
class="math inline">\(B\)</span> depends on whether we measure the
distance from <span class="math inline">\(A\)</span> to <span
class="math inline">\(B\)</span> or from <span
class="math inline">\(B\)</span> to <span
class="math inline">\(A\)</span>. That is to say that the distance from
<span class="math inline">\(A\)</span> to <span
class="math inline">\(B\)</span> isn’t the same as the distance from
<span class="math inline">\(B\)</span> to <span
class="math inline">\(A\)</span>.</p>
<p>There are two ways in which the rate of nucleotide substitution can
be allowed to vary from position to positionthe phenomenon of among-site
rate variation. First, we expect the rate of substitution to depend on
codon position in protein-coding genes. The sequence can be divided into
first, second, and third codon positions and rates calculated separately
for each of those positions. Second, we can assume <span><em>a
priori</em></span> that there is a distribution of different rates
possible and that this distribution is described by one of the standard
distributions from probability theory. We then imagine that the
substitution rate at any given site is determined by a random draw from
that probability distribution. The gamma distribution is widely to
describe the pattern of among-site rate variation, because it can
approximate a wide variety of different distributions (Figure <a
href="#fig:asrv" data-reference-type="ref"
data-reference="fig:asrv">1</a>).<a href="#fn16" class="footnote-ref"
id="fnref16" role="doc-noteref"><sup>16</sup></a></p>
<figure id="fig:asrv">
<div class="center">
<img src="gamma.png" />
</div>
<figcaption>Examples of a gamma distribution.</figcaption>
</figure>
<p>The mean substitution rate in each curve above is 0.1. The curves
differ only in the value of a parameter, <span
class="math inline">\(\alpha\)</span>, called the “shape parameter.” The
shape parameter gives a nice numerical description of how much rate
variation there is, except that it’s backwards. The larger the
parameter, the less among-site rate variation there is.</p>
<h1 class="unnumbered"
id="the-neutral-theory-of-molecular-evolution">The neutral theory of
molecular evolution</h1>
<p>I didn’t make a big deal of it in what we just went over, but in
deriving the Jukes-Cantor equation I used the phrase “substitution rate”
instead of the phrase “mutation rate.”<a href="#fn17"
class="footnote-ref" id="fnref17" role="doc-noteref"><sup>17</sup></a>
As a preface to what is about to follow, let me explain the
difference.</p>
<ul>
<li><p><span><em>Mutation rate</em></span> refers to the rate at which
changes are incorporated into a nucleotide sequence during the process
of replication, i.e., the probability that an allele differs from the
copy of that allele in its parent from which it was derived. <span>
<em>Mutation rate</em></span> refers to the rate at which mutations
arise.</p></li>
<li><p>An allele substitution occurs when a newly arisen allele is
incorporated into a population, e.g., when a newly arisen allele becomes
fixed in a population. <span><em>Substitution rate</em></span> refers to
the rate at which allele substitutions occur.</p></li>
</ul>
<p>Mutation rates and substitution rates are obviously
relatedsubstitutions can’t happen unless mutations occur, after all, but
it’s important to remember that they refer to different processes.</p>
<h1 class="unnumbered" id="early-empirical-observations">Early empirical
observations</h1>
<p>By the early 1960s amino acid sequences of hemoglobins and cytochrome
<span><em>c</em></span> for many mammals had been determined. When the
sequences were compared, investigators began to notice that the number
of amino acid differences between different pairs of mammals seemed to
be roughly proportional to the time since they had diverged from one
another, as inferred from the fossil record. Zuckerkandl and
Pauling <span class="citation"
data-cites="Zuckerkandl-Pauling65"></span> proposed the
<span><em>molecular clock hypothesis</em></span> to explain these
results. Specifically, they proposed that there was a constant rate of
amino acid substitution over time. Sarich and Wilson <span
class="citation" data-cites="Sarich-Wilson67 Wilson-Sarich69"></span>
used the molecular clock hypothesis to propose that humans and apes
diverged approximately 5 million years ago. While that proposal may not
seem particularly controversial now, it generated enormous controversy
at the time, because at the time many paleoanthropologists interpreted
the evidence to indicate humans diverged from apes as much as 30 million
years ago.</p>
<p>One year after Zuckerkandl and Pauling’s paper, Harris <span
class="citation" data-cites="Harris66"></span> and Hubby and
Lewontin <span class="citation"
data-cites="Hubby-Lewontin66 Lewontin-Hubby66"></span> showed that
protein electrophoresis could be used to reveal surprising amounts of
genetic variability within populations. Harris studied 10 loci in human
populations, found three of them to be polymorphic, and identified one
locus with three alleles. Hubby and Lewontin studied 18 loci in
<span><em>Drosophila pseudoobscura</em></span>, found seven to be
polymorphic, and five that had three or more alleles.</p>
<p>Both sets of observations posed real challenges for evolutionary
geneticists. It was difficult to imagine an evolutionary mechanism that
could produce a constant rate of substitution. It was similarly
difficult to imagine that natural selection could maintain so much
polymorphism within populations. The “cost of selection,” as
Haldane <span class="citation" data-cites="Haldane-1957"></span> called
it would simply be too high.</p>
<h1 class="unnumbered"
id="neutral-substitutions-and-neutral-variation">Neutral substitutions
and neutral variation</h1>
<p>Kimura <span class="citation" data-cites="Kimura68"></span> and King
and Jukes <span class="citation" data-cites="King-Jukes69"></span>
proposed a way to solve both empirical problems. If the vast majority of
amino acid substitutions are selectively neutral,<a href="#fn18"
class="footnote-ref" id="fnref18" role="doc-noteref"><sup>18</sup></a>
then substitutions will occur at approximately a constant rate (assuming
that substitution rates don’t vary over time) and it will be easy to
maintain lots of polymorphism within populations because there will be
no cost of selection. I’ll develop both of those points in a bit more
detail in just a moment, but let me first be precise about what the
neutral theory of molecular evolution actually proposes. More
specifically, let me first be precise about what it does
<span><em>not</em></span> propose. I’ll do so specifically in the
context of protein evolution for now, although we’ll broaden the scope
later.</p>
<ul>
<li><p><span><em>The neutral theory asserts that alternative alleles at
variable protein loci are selectively neutral.</em></span> This does
<span> <em>not</em></span> mean that the locus is unimportant, only that
the alternative alleles found at this locus are selectively neutral.</p>
<ul>
<li><p>Glucose-phosphate isomerase is an esssential enzyme. It catalyzes
the first step of glycolysis, the conversion of glucose-6-phosphate into
fructose-6-phosphate.</p></li>
<li><p>Natural populations of many, perhaps most, populations of plants
and animals are polymorphic at this locus, i.e., they have two or more
alleles with different amino acid sequences.</p></li>
<li><p>The neutral theory asserts that the alternative alleles are
essentially equivalent in fitness, in the sense that genetic drift,
rather than natural selection, dominates the dynamics of frequency
changes among them.</p></li>
</ul></li>
<li><p>By <span><em>selectively neutral</em></span> we do
<span><em>not</em></span> mean that the alternative alleles have no
effect on physiology or fitness. We mean that the selection among
different genotypes at this locus is sufficiently weak that the pattern
of variation is determined primarily by the interaction of mutation,
drift, mating system, and migration. This is roughly equivalent to
saying that <span class="math inline">\(N_es &lt; 1\)</span>, where
<span class="math inline">\(N_e\)</span> is the effective population
size and <span class="math inline">\(s\)</span> is the selection
coefficient on alleles at this locus.</p>
<ul>
<li><p>Experiments in <span><em>Colias</em></span> butterflies, and
other organisms have shown that different electrophoretic variants of
GPI have different enzymatic capabilities and different thermal
stabilities. In some cases, these differences have been related to
differences in individual performance.</p></li>
<li><p>If populations of <span><em>Colias</em></span> are large and the
differences in fitness associated with differences in genotype are
large, i.e., if <span class="math inline">\(N_es &gt; 1\)</span>, then
selection plays a predominant role in determining patterns of diversity
at this locus, i.e., the neutral theory of molecular evolution would not
apply.</p></li>
<li><p>If populations of <span><em>Colias</em></span> are small or the
differences in fitness associated with differences in genotype are
small, or both, then drift plays a predominant role in determining
patterns of diversity at this locus, i.e., the neutral theory of
molecular evolution applies.</p></li>
</ul></li>
</ul>
<p>In short, the neutral theory of molecular really asserts only that
observed amino acid substitutions and polymorphisms are <span>
<em>effectively</em></span> neutral, not that the loci involved are
unimportant or that allelic differences at those loci have no effect on
fitness.</p>
<h2 class="unnumbered" id="the-rate-of-molecular-evolution">The rate of
molecular evolution</h2>
<p>We’re now going to calculate the rate of molecular evolution, i.e.,
the rate of allelic substitution, under the hypothesis that mutations
are selectively neutral.<a href="#fn19" class="footnote-ref"
id="fnref19" role="doc-noteref"><sup>19</sup></a> To get that rate we
need two things: the rate at which new mutations occur and the
probability with which new mutations are fixed. In a word equation <span
class="math display">\[\begin{aligned}
\mbox{\# of substitutions/generation} &amp;=&amp; (\mbox{\# of
mutations/generation})\times(\mbox{probability
  of fixation}) \\
\lambda &amp;=&amp; \mu_0p_0 \quad .
\end{aligned}\]</span> Surprisingly,<a href="#fn20" class="footnote-ref"
id="fnref20" role="doc-noteref"><sup>20</sup></a> it’s pretty easy to
calculate both <span class="math inline">\(\mu_0\)</span> and <span
class="math inline">\(p_0\)</span> from first principles.</p>
<p>In a diploid population of size <span
class="math inline">\(N\)</span>, there are <span
class="math inline">\(2N\)</span> gametes. The probability that any one
of them mutates is just the mutation rate, <span
class="math inline">\(\mu\)</span>, so <span
class="math display">\[\mu_0 = 2N\mu \quad . \label{eq:mu-0}\]</span> To
calculate the probability of fixation, we have to say something about
the dynamics of alleles in populations. Let’s suppose that we’re dealing
with a single population, to keep things simple. Now, you have to
remember a little of what you learned about the properties of genetic
drift. If the current frequency of an allele is <span
class="math inline">\(p_0\)</span>, what’s the probability that is
eventually fixed? <span class="math inline">\(p_0\)</span>. When a new
mutation occurs there’s only one copy of it,<a href="#fn21"
class="footnote-ref" id="fnref21" role="doc-noteref"><sup>21</sup></a>
so the frequency of a newly arisen mutation is <span
class="math inline">\(1/2N\)</span> and <span class="math display">\[p_0
= \frac{1}{2N} \quad . \label{eq:p-0}\]</span> Putting (<a
href="#eq:mu-0" data-reference-type="ref"
data-reference="eq:mu-0">[eq:mu-0]</a>) and (<a href="#eq:p-0"
data-reference-type="ref" data-reference="eq:p-0">[eq:p-0]</a>) together
we find <span class="math display">\[\begin{aligned}
\lambda &amp;=&amp; \mu_0p_0 \\
        &amp;=&amp; (2N\mu)\left(\frac{1}{2N}\right) \\
        &amp;=&amp; \mu \quad .
\end{aligned}\]</span> In other words, if mutations are selectively
neutral, the substitution rate is equal to the mutation rate. Since
mutation rates are (mostly) governed by physical factors that remain
relatively constant, mutation rates should remain constant, implying
that substitution rates should remain constant if substitutions are
selectively neutral. In short, if mutations are selectively neutral, we
expect a molecular clock.</p>
<h2 class="unnumbered" id="diversity-in-populations">Diversity in
populations</h2>
<p>Protein-coding genes consist of hundreds or thousands of nucleotides,
each of which could mutate to one of three other nucleotides.<a
href="#fn22" class="footnote-ref" id="fnref22"
role="doc-noteref"><sup>22</sup></a> That’s not an infinite number of
possibilities, but it’s pretty large.<a href="#fn23"
class="footnote-ref" id="fnref23" role="doc-noteref"><sup>23</sup></a>
It suggests that we could treat every mutation that occurs as if it were
completely new, a mutation that has never been seen before and will
never be seen again. Does that description ring any bells? Does the
infinite alleles model sound familiar? It should, because it exactly
fits the situation I’ve just described.</p>
<p>Having remembered that this situation is well described by the
infinite alleles model, I’m sure you’ll also remember that we can
calculate the equilibrium inbreeding coefficient for the infinite
alleles model, i.e., <span class="math display">\[f = \frac{1}{4N_e\mu +
1} \quad .\]</span> What’s important about this for our purposes, is
that to the extent that the infinite alleles model is appropriate for
molecular data, then <span class="math inline">\(f\)</span> is the
frequency of homozygotes we should see in populations and <span
class="math inline">\(1-f\)</span> is the frequency of heterozygotes. So
in large populations we should find more diversity than in small ones,
which is roughly what we do find. Notice, however, that here we’re
talking about heterozygosity at individual nucleotide positions,<a
href="#fn24" class="footnote-ref" id="fnref24"
role="doc-noteref"><sup>24</sup></a> not heterozygosity of
halpotypes.</p>
<h1 class="unnumbered" id="conclusions">Conclusions</h1>
<p>In broad outline then, the neutral theory does a pretty good job of
dealing with at least some types of molecular data. I’m sure that some
of you are already thinking, “But what about third codon positions
<span><em>versus</em></span> first and second?” or “What about the
observation that histone loci evolve much more slowly than interferons
or MHC loci?” Those are good questions, and those are where we’re going
next. As we’ll see, molecular evolutionists have elaborated the
framework extensively<a href="#fn25" class="footnote-ref" id="fnref25"
role="doc-noteref"><sup>25</sup></a> in the last sixty years, but these
basic principles underlie every investigation that’s conducted. That’s
why I wanted to spend a fair amount of time going over the logic and
consequences. Besides, it’s a rare case in population genetics where the
fundamental mathematics that lies behind some important predictions are
easy to understand.<a href="#fn26" class="footnote-ref" id="fnref26"
role="doc-noteref"><sup>26</sup></a></p>
<h1 class="unnumbered" id="creative-commons-license">Creative Commons
License</h1>
<p>These notes are licensed under the Creative Commons Attribution
License. To view a copy of this license, visit or send a letter to
Creative Commons, 559 Nathan Abbott Way, Stanford, California 94305,
USA.</p>
<aside id="footnotes" class="footnotes footnotes-end-of-document"
role="doc-endnotes">
<hr />
<ol>
<li id="fn1"><p>In certain cases it may make sense to talk about a
phylogeny of populations within species, but in many cases it doesn’t.
We’ll discuss this further when we get to phylogeography in a couple of
weeks.<a href="#fnref1" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn2"><p>This approach is much less common than it used to be. In
the “old days” (meaning when I was a young assistant professor), we had
vigorous debates about whether or not it was reasonable to incorporate
some knowledge of evolutionary processes into the methods we use for
inferring evolutionary patterns. Now it’s pretty much taken for granted
that we should. One way of justifying a strict parsimony approach to
cladistics, however, is by arguing (a) that by minimizing character
state changes on a tree you’re merely trying to find a pattern of
character changes as consistent as possible with the data you’ve
gathered and (b) that evolutionary processes should be invoked only to
explain that pattern, not to construct it.<a href="#fnref2"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn3"><p>Of course, if you really care about making inferences
about evollutionary patterns from molecular data, especially patterns
above the species level, you’ll want to take the course Paul Lewis
teaches. He spends pretty much the entire course discussing these
problems.<a href="#fnref3" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn4"><p>Of course, differences in expression must ultimately be
the result either of a DNA sequence difference somewhere, e.g., in a
promoter sequence or the locus encoding a promotor or repressor protein,
if it is a genetic difference or of an epigenetic modification of the
sequence, e.g., by methylation.<a href="#fnref4" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn5"><p>Recent evidence suggests that mitochondria may
occasionaly be inherited biparentally in humans <span class="citation"
data-cites="Luo-etal-2018"></span>.<a href="#fnref5"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn6"><p>Note: Several of the techniques in this list are
primarily of historical interest. They were widely used in the past, but
they are no longer used (or no longer used very much).<a href="#fnref6"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn7"><p>See below for a description of some of these repetitive
seqeuences.<a href="#fnref7" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn8"><p>Classical Mendelian genetics, and quantitative genetics
too for that matter, depends on genetic variation in traits to identify
the presence of a gene.<a href="#fnref8" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn9"><p>To be fair, it is possible to distinguish heterozygotes
from homozyotes with AFLPs, if you are
<span><em><strong>very</strong></em></span> careful with your PCR
technique <span class="citation" data-cites="Jansen-etal-2001"></span>.
That being said, few people are careful enough with their PCR to be able
to score AFLPs reliably as codominant markers, and I am unaware of
anyone who has done so outside of a controlled breeding program.<a
href="#fnref9" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn10"><p>The rapidly diminishing cost of high-throughput
nucleotide sequencing, however, suggests that microsatellites will soon
join allozymes, RAPDs, AFLPs, ISSRs, and RFLPs as of interest primarily
for historical reasons.<a href="#fnref10" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn11"><p>In the old days, sequencing DNA meant running samples
on a polyacrylamide gel, transferring them to a membrane, hybridizing
with <span class="math inline">\(^{32}P\)</span>, and exposing X-ray
film to the membrane for several days before developing it.<a
href="#fnref11" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn12"><p>For example, Nora Mitchell’s paper on the phylogeny of
<span> <em>Protea</em></span> <span class="citation"
data-cites="Mitchell-etal-2017"></span> was based on analysis of
nucleotide sequence variation at nearly 500 loci.<a href="#fnref12"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn13"><p>By now you should realize that when I write that
something is “fairly simple”, I mean that it’s fairly simple to someone
who’s comfortable with mathematics.<a href="#fnref13"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn14"><p>Notice that I wrote “substitution,” not “mutation.”
We’ll come back to this distinction later. It turns out to be really
important.<a href="#fnref14" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn15"><p>The factor 2 is there because <span
class="math inline">\(\lambda t\)</span> substitutions are expected on
each branch. In fact, you will usually see the equation for <span
class="math inline">\(q_t\)</span> written as <span
class="math inline">\(q_t = 1 - (3/4)\left(1 - e^{-4\alpha
t/3}\right)\)</span>, where <span class="math inline">\(\alpha =
2\lambda\)</span>. <span class="math inline">\(\alpha\)</span> is also
referred to as the substitution rate, but it refers to the rate of
substitution between the two sequences, not to the rate of substitution
between each sequence and their common ancestor.<a href="#fnref15"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn16"><p>And, to be honest, because it is mathematically
convenient to work with.<a href="#fnref16" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn17"><p>In fact, I just mentioned the distinction in passing in
two different footnotes.<a href="#fnref17" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn18"><p>Notice that I just said that we’re going to assume that
the vast majority of nucleotide <span><em>substitutions</em></span> are
selectively neutral. This doesn’t mean that most nucleotide
<span><em>mutations</em></span> are selectively neutral. Indeed, we’ll
see that most of them are deleterious.<a href="#fnref18"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn19"><p>Notice that contrary to what I said earlier, here I am
assuming that <span><em>mutations</em></span> are neutral, not just
substitutions.<a href="#fnref19" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn20"><p>Or perhaps not.<a href="#fnref20" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn21"><p>By definition. It’s new.<a href="#fnref21"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn22"><p>Why three when there are four nucleotides? Because if
the nucleotide at a certain position is an A, for example, it can only
<span><em>change</em></span> to a C, G, or T.<a href="#fnref22"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn23"><p>If a protein consists of 400 amino acids, that’s 1200
nucleotides. There are <span class="math inline">\(4^{1200} \approx
10^{720}\)</span> different sequences that are 1200 nucleotides long.
For context, there are only about <span class="math inline">\(3.28
\times 10^{80}\)</span> elementary particles in the universe (<a
href="https://www.popularmechanics.com/space/a27259/how-many-particles-are-in-the-entire-universe/"
class="uri">https://www.popularmechanics.com/space/a27259/how-many-particles-are-in-the-entire-universe/</a>).<a
href="#fnref23" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn24"><p>Since the mutation rate we’re talking about applies to
individual nucleotide positions.<a href="#fnref24" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn25"><p>That mean’s they’ve made it more complicated.<a
href="#fnref25" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn26"><p>It’s the concepts that get tricky, not the algebra, or
at least that’s what I think.<a href="#fnref26" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
</ol>
</aside>
</body>
</html>
