<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Estimating viability</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    /* The extra [class] is a hack that increases specificity enough to
       override a similar rule in reveal.js */
    ul.task-list[class]{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      font-size: inherit;
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
  </style>
  <link rel="stylesheet" href="pandoc.css" />
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script
  src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js"
  type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Estimating viability</h1>
</header>
<nav id="TOC" role="doc-toc">
<ul>
<li><a href="#introduction" id="toc-introduction">Introduction</a></li>
<li><a href="#estimating-relative-viability"
id="toc-estimating-relative-viability">Estimating relative
viability</a></li>
<li><a href="#an-example" id="toc-an-example">An example</a></li>
<li><a href="#creative-commons-license"
id="toc-creative-commons-license">Creative Commons License</a></li>
</ul>
</nav>
<h1 class="unnumbered" id="introduction">Introduction</h1>
<p>Being able to make predictions with known (or estimated) viabilities,
doesn’t do us a heck of a lot of good unless we can figure out what
those viabilities are. Fortunately, figuring them out isn’t too hard.<a
href="#fn1" class="footnote-ref" id="fnref1"
role="doc-noteref"><sup>1</sup></a> If we know the number of individuals
of each genotype before selection, it’s really easy as a matter of
fact.<a href="#fn2" class="footnote-ref" id="fnref2"
role="doc-noteref"><sup>2</sup></a> Consider that our data looks like
this:</p>
<div class="center">
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Genotype</th>
<th style="text-align: center;"><span
class="math inline">\(A_1A_1\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(A_1A_2\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(A_2A_2\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Number in zygotes</td>
<td style="text-align: center;"><span
class="math inline">\(n_{11}^{(z)}\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(n_{12}^{(z)}\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(n_{22}^{(z)}\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">Viability</td>
<td style="text-align: center;"><span
class="math inline">\(w_{11}\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(w_{12}\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(w_{22}\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;">Number in adults</td>
<td style="text-align: center;"><span class="math inline">\(n_{11}^{(a)}
= w_{11}n_{11}^{(z)}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(n_{12}^{(a)}
= w_{12}n_{12}^{(z)}\)</span></td>
<td style="text-align: center;"><span class="math inline">\(n_{22}^{(a)}
= w_{22}n_{22}^{(z)}\)</span></td>
</tr>
</tbody>
</table>
</div>
<p>In other words, estimating the absolute viability simply consists of
estimating the probability that an individuals of each genotype that
survive from zygote to adult. The maximum-likelihood estimate is, of
course, just what you would probably guess: <span
class="math display">\[w_{ij} = \frac{n_{ij}^{(a)}}{n_{ij}^{(z)}} \quad
,\]</span> Since <span class="math inline">\(w_{ij}\)</span> is a
probability and the outcome is binary (survive or die), you should be
able to guess what kind of likelihood relates the observed data to the
unseen parameter, namely, a binomial likelihood. In
<span><code>Stan</code></span> notation:<a href="#fn3"
class="footnote-ref" id="fnref3" role="doc-noteref"><sup>3</sup></a></p>
<pre><code>   n_11_adult ~ binomial(n_11_zygote, w_11)
   n_12_adult ~ binomial(n_12_zygote, w_12)
   n_22_adult ~ binomial(n_22_zygote, w_22)</code></pre>
<h1 class="unnumbered" id="estimating-relative-viability">Estimating
relative viability</h1>
<p>To estimate absolute viabilities, we have to be able to identify
genotypes non-destructively, because we have to know what their genotype
was both <span><em>before</em></span> the selection event and
<span><em>after</em></span> the selection event. That’s fine if we
happen to be dealing with an experimental situation where we can do
controlled crosses to establish known genotypes or if we happen to be
studying an organism and a trait where we can identify the genotype from
the phenotype of a zygote (or at least a very young individual) and from
surviving adults.<a href="#fn4" class="footnote-ref" id="fnref4"
role="doc-noteref"><sup>4</sup></a> What do we do when we can’t follow
the survival of individuals with known genotype? Give up?<a href="#fn5"
class="footnote-ref" id="fnref5" role="doc-noteref"><sup>5</sup></a></p>
<p>Remember that to make inferences about how selection will act, we
only need to know <span><em>relative</em></span> viabilities, not
<span><em>absolute</em></span> viabilities.<a href="#fn6"
class="footnote-ref" id="fnref6" role="doc-noteref"><sup>6</sup></a> We
still need to know something about the genotypic composition of the
population before selection, but it turns out that if we’re only
interested in relative viabilities, we don’t need to follow individuals.
All we need to be able to do is to score genotypes and estimate genotype
frequencies before and after selection. Our data looks like this:</p>
<div class="center">
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Genotype</th>
<th style="text-align: center;"><span
class="math inline">\(A_1A_1\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(A_1A_2\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(A_2A_2\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Frequency in zygotes</td>
<td style="text-align: center;"><span
class="math inline">\(x_{11}^{(z)}\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(x_{12}^{(z)}\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(x_{22}^{(z)}\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">Frequency in adults</td>
<td style="text-align: center;"><span
class="math inline">\(x_{11}^{(a)}\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(x_{12}^{(a)}\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(x_{22}^{(a)}\)</span></td>
</tr>
</tbody>
</table>
</div>
<p>We also know that <span class="math display">\[\begin{aligned}
x_{11}^{(a)} &amp;=&amp; w_{11}x_{11}^{(z)}/\bar w \\
x_{12}^{(a)} &amp;=&amp; w_{12}x_{12}^{(z)}/\bar w \\
x_{22}^{(a)} &amp;=&amp; w_{22}x_{22}^{(z)}/\bar w \quad .
\end{aligned}\]</span> Suppose we now divide all three equations by the
middle one: <span class="math display">\[\begin{aligned}
x_{11}^{(a)}/x_{12}^{(a)} &amp;=&amp;
w_{11}x_{11}^{(z)}/w_{12}x_{12}^{(z)} \\
1 &amp;=&amp; 1 \\
x_{22}^{(a)}/x_{12}^{(a)} &amp;=&amp;
w_{22}x_{22}^{(z)}/w_{12}x_{12}^{(z)} \quad ,
\end{aligned}\]</span> or, rearranging a bit <span
class="math display">\[\begin{aligned}
\frac{w_{11}}{w_{12}} &amp;=&amp;
\left(\frac{x_{11}^{(a)}}{x_{12}^{(a)}}\right)
                          \left(\frac{x_{12}^{(z)}}{x_{11}^{(z)}}\right)
\label{eq:est-rel-viability-1}
\\
\frac{w_{22}}{w_{12}} &amp;=&amp;
\left(\frac{x_{22}^{(a)}}{x_{12}^{(a)}}\right)
                          \left(\frac{x_{12}^{(z)}}{x_{22}^{(z)}}\right)
\quad . \label{eq:est-rel-viability-2}
\end{aligned}\]</span> This gives us a complete set of relative
viabilities.</p>
<div class="center">
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Genotype</th>
<th style="text-align: center;"><span
class="math inline">\(A_1A_1\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(A_1A_2\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(A_2A_2\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">Relative viability</td>
<td style="text-align: center;"><span
class="math inline">\(\frac{w_{11}}{w_{12}}\)</span></td>
<td style="text-align: center;">1</td>
<td style="text-align: center;"><span
class="math inline">\(\frac{w_{22}}{w_{12}}\)</span></td>
</tr>
<tr class="odd">
<td style="text-align: left;"></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
<td style="text-align: center;"></td>
</tr>
</tbody>
</table>
</div>
<p>If we use the maximum-likelihood estimates for genotype frequencies
before and after selection, we obtain maximum likelihood estimates for
the relative viabilities.<a href="#fn7" class="footnote-ref" id="fnref7"
role="doc-noteref"><sup>7</sup></a> If we use Bayesian methods to
estimate genotype frequencies before and after selection (including the
uncertainty around those estimates), we can use these formulas to get
Bayesian estimates of the relative viabilities (and the uncertainty
around them).</p>
<h1 class="unnumbered" id="an-example">An example</h1>
<p>Let’s see how this works with some real data from Dobzhansky’s work
on chromosome inversion polymorphisms in <span><em>Drosophila
pseudoobscura.</em></span><a href="#fn8" class="footnote-ref"
id="fnref8" role="doc-noteref"><sup>8</sup></a></p>
<div class="center">
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Genotype</th>
<th style="text-align: center;"><span
class="math inline">\(ST/ST\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(ST/CH\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(CH/CH\)</span></th>
<th style="text-align: center;">Total</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Number in larvae</td>
<td style="text-align: center;">41</td>
<td style="text-align: center;">82</td>
<td style="text-align: center;">27</td>
<td style="text-align: center;">150</td>
</tr>
<tr class="even">
<td style="text-align: left;">Number in adults</td>
<td style="text-align: center;">57</td>
<td style="text-align: center;">169</td>
<td style="text-align: center;">29</td>
<td style="text-align: center;">255</td>
</tr>
</tbody>
</table>
</div>
<p>You may be wondering how the sample of adults can be larger than the
sample of larvae. That’s because to score an individual’s inversion
type, Dobzhansky had to kill it. The numbers in larvae are based on a
sample of the population, and the adults that survived were not
genotyped as larvae. As a result, all we can do is to estimate the
relative viabilities. <span class="math display">\[\begin{aligned}
\frac{w_{11}}{w_{12}} &amp;=&amp;
\left(\frac{x_{11}^{(a)}}{x_{12}^{(a)}}\right)
                          \left(\frac{x_{12}^{(z)}}{x_{11}^{(z)}}\right)
= \left(\frac{57/255}{169/255}\right)
  \left(\frac{82/150}{41/150}\right)
= 0.67 \\
\frac{w_{22}}{w_{12}} &amp;=&amp;
\left(\frac{x_{22}^{(a)}}{x_{12}^{(a)}}\right)
                          \left(\frac{x_{12}^{(z)}}{x_{22}^{(z)}}\right)
= \left(\frac{29/255}{169/255}\right)
  \left(\frac{82/150}{27/150}\right)
= 0.52 \quad .
\end{aligned}\]</span> So it looks as if we have balancing selection,
i.e., the fitness of the heterozygote exceeds that of either
homozygote.</p>
<p>We can check to see whether this conclusion is statistically
justified by comparing the observed number of individuals in each
genotype category in adults with what we’d expect if all genotypes were
equally likely to survive.</p>
<div class="center">
<table>
<thead>
<tr class="header">
<th style="text-align: left;">Genotype</th>
<th style="text-align: center;"><span
class="math inline">\(ST/ST\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(ST/CH\)</span></th>
<th style="text-align: center;"><span
class="math inline">\(CH/CH\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Expected</td>
<td style="text-align: center;"><span
class="math inline">\(\left(\frac{41}{150}\right)255\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(\left(\frac{82}{150}\right)255\)</span></td>
<td style="text-align: center;"><span
class="math inline">\(\left(\frac{27}{150}\right)255\)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;"></td>
<td style="text-align: center;">69.7</td>
<td style="text-align: center;">139.4</td>
<td style="text-align: center;">45.9</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Observed</td>
<td style="text-align: center;">57</td>
<td style="text-align: center;">169</td>
<td style="text-align: center;">29</td>
</tr>
<tr class="even">
<td colspan="4" style="text-align: left;"><span
class="math inline">\(\chi^2_2 = 14.82\)</span>, <span
class="math inline">\(P &lt; 0.001\)</span></td>
</tr>
</tbody>
</table>
</div>
<p>So we have strong evidence that genotypes differ in their probability
of survival.</p>
<p>We can also use our knowledge of how selection works to predict the
genotype frequencies at equilibrium: <span
class="math display">\[\begin{aligned}
\frac{w_{11}}{w_{12}} &amp;=&amp; 1 - s_1 \\
\frac{w_{22}}{w_{12}} &amp;=&amp; 1 - s_2 \quad .
\end{aligned}\]</span> So <span class="math inline">\(s_1 =
0.33\)</span>, <span class="math inline">\(s_2 = 0.48\)</span>, and the
predicted equilibrium frequency of the <span
class="math inline">\(ST\)</span> chromosome is <span
class="math inline">\(s_2/(s_1+s_2) = 0.59\)</span>.</p>
<h1 class="unnumbered" id="creative-commons-license">Creative Commons
License</h1>
<p>These notes are licensed under the Creative Commons Attribution
License. To view a copy of this license, visit or send a letter to
Creative Commons, 559 Nathan Abbott Way, Stanford, California 94305,
USA.</p>
<aside id="footnotes" class="footnotes footnotes-end-of-document"
role="doc-endnotes">
<hr />
<ol>
<li id="fn1"><p>I almost said that it was easy, but that would be going
a bit too far.<a href="#fnref1" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn2"><p>And in the very next sentence I contradicted the last
footnote. But it really is easy to estimate viabilities if we can
genotype individuals before and after selection.<a href="#fnref2"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn3"><p>You knew you were going to see this again, didn’t you?<a
href="#fnref3" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn4"><p>How many organisms and traits can you think of that
satisfy this criterion? Any? There is one other possibility: If we can
identify an individual’s genotype after it’s dead
<span><em>and</em></span> if we can construct a random sample that
includes both living and dead individuals <span><em>and</em></span> if
we assume the probability of including an individual in the sample
doesn’t depend on whether that individual is dead or alive, then we can
sample a population after the selection event and score genotypes both
before and after the event from one set of observations.<a
href="#fnref4" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn5"><p>Would I be asking the question if the answer were
“Yes”?<a href="#fnref5" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn6"><p>At least that’s true until we start worrying about how
selection and drift interact.<a href="#fnref6" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn7"><p>If anyone cares, it’s because of the invariance property
of maximum-likelihod estimates. If you don’t understand what that is,
don’t worry about it, just trust me. Or if you want to know what the
invariance principle is, ask.<a href="#fnref7" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn8"><p>Taken from <span class="citation"
data-cites="Dobzhansky-1947"></span>.<a href="#fnref8"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</aside>
</body>
</html>
