<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Statistical phylogeography: Migrate-N, IMa, and ABC</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    /* The extra [class] is a hack that increases specificity enough to
       override a similar rule in reveal.js */
    ul.task-list[class]{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      font-size: inherit;
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
  </style>
  <link rel="stylesheet" href="pandoc.css" />
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script
  src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js"
  type="text/javascript"></script>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Statistical phylogeography: Migrate-N, IMa, and
ABC</h1>
</header>
<nav id="TOC" role="doc-toc">
<ul>
<li><a href="#gene-trees-versus-population-trees"
id="toc-gene-trees-versus-population-trees">Gene trees
<span><em>versus</em></span> population trees</a></li>
<li><a href="#coalescent-based-estimates-of-migration-rate"
id="toc-coalescent-based-estimates-of-migration-rate">Coalescent-based
estimates of migration rate</a></li>
<li><a href="#divergence-and-migration"
id="toc-divergence-and-migration">Divergence and migration</a>
<ul>
<li><a href="#an-example" id="toc-an-example">An example</a></li>
<li><a href="#extending-the-approach-to-multiple-populations"
id="toc-extending-the-approach-to-multiple-populations">Extending the
approach to multiple populations</a></li>
</ul></li>
<li><a href="#phylogeography-of-montane-grasshoppers"
id="toc-phylogeography-of-montane-grasshoppers">Phylogeography of
montane grasshoppers</a></li>
<li><a href="#approximate-bayesian-computation-motivation"
id="toc-approximate-bayesian-computation-motivation">Approximate
Bayesian computation: motivation</a></li>
<li><a href="#an-example-1" id="toc-an-example-1">An example</a></li>
<li><a href="#approximate-bayesian-computation"
id="toc-approximate-bayesian-computation">Approximate Bayesian
Computation</a></li>
<li><a href="#back-to-the-real-world"
id="toc-back-to-the-real-world">Back to the real world</a></li>
<li><a href="#limitations-of-abc"
id="toc-limitations-of-abc">Limitations of ABC</a></li>
<li><a href="#creative-commons-license"
id="toc-creative-commons-license">Creative Commons License</a></li>
</ul>
</nav>
<p>As we’ve seen in our discussion of AMOVA, nucleotide sequence data
seem to have the potential to help us make not only evolutionary
inferences about the sequences, but also about the evolutionary history
of the populations from which we’ve collected those sequences. In the
mid-1990s Alan Templeton introduced nested clade analysis <span
class="citation"
data-cites="Templeton-2004 Templeton-2009 Templeton-etal-1995"></span>
as a formal approach to using an estimate of phylogenetic relationships
among haplotypes to infer something both about the biogeographic history
of the populations in which they are contained and the evolutionary
processes associated with the pattern of diversification implied by the
phylogenetic relationships among haplotypes and their geographic
distribution. The statistical parsimony part of NCA depends heavily on
coalescent theory for calculating the “limits” of parsimony. As a
result, NCA combines aspects of pure phylogenetic inferenceparsimonywith
aspects of pure population geneticscoalescent theoryto develop a set of
inferences about the phylogeographic history of populations within
species. So far as I am aware, no one uses NCA any more, but it is
important to mention it as an early attempt to formalize the process of
inferring the evolutionary history of populations from nucleotide
sequence data. Prior to Templeton the process of inference was really
just storytelling, storytelling that made a reasonablle amount of sense,
but still storytelling. Now everyone uses methods based directly on
coalescent theory or similar approaches. Before we get to that, though,
I need to describe one complication that is taken for granted now that
first became widely recognized in the late 1980s. Pekka Pamilo and
Mashatoshi Nei <span class="citation"
data-cites="Pamilo-Nei-1988"></span> pointed out that the phylogenetic
relationships of a single gene might be different from those of the
populations from which the samples were collected.</p>
<h1 class="unnumbered" id="gene-trees-versus-population-trees">Gene
trees <span><em>versus</em></span> population trees</h1>
<p>Gene trees describe the evolutionary relationships, i.e., the
phylogeny of a set of genes. Population trees describe the phylogeny of
a set of populations. We often want to infer the history of populations
from a set of genes that we collect from those populations. There are
several reasons why <span><em>gene trees</em></span> might not match
<span><em>population trees</em></span>.</p>
<ul>
<li><p>It could simply be a problem of estimation. Given a particular
set of gene sequences, we <span><em>estimate</em></span> a phylogenetic
relationship among them. But our estimate could be wrong. In fact, given
the astronomical number of different trees possible with 50 or 60
distinct sequences, every phylogenetic estimate is virtually certain to
be wrong somewhere. We just don’t know where. So a difference between
our <span><em>estimate</em></span> of a gene tree and the population
tree could mean nothing more than that they actually match, but our gene
tree estimate is wrong.</p></li>
<li><p>There might have been a hybridization event in the past so that
the phylogenetic history of the gene we’re studying is different from
that of the populations from which we sampled. Hybridization is
especially likely to have a large impact if the locus for which we have
information is uniparentally inherited, e.g., mitochondrial or
chloroplast DNA. A single hybridization event in the distant past in
which the maternal parent was from a different population will give
mtDNA or cpDNA a very different phylogeny than nuclear genes that
underwent a lot of backcrossing after the hybridization event.</p></li>
<li><p>If the ancestral population was polymorphic at the time the
initial split occurred alleles that are more distantly related might, by
chance, end up in the same descendant population (see Figure <a
href="#fig:ancestral-polymorphism" data-reference-type="ref"
data-reference="fig:ancestral-polymorphism">1</a>)</p></li>
</ul>
<figure id="fig:ancestral-polymorphism">
<div class="center">
<img src="ancestral-polymorphism.png" style="height:10cm" />
</div>
<figcaption>Discordance between gene and population trees as a result of
ancestral polymorphism (from <span class="citation"
data-cites="Knowles-2001"></span>).</figcaption>
</figure>
<p>As Pamilo and Nei showed, it’s possible to calculate the probability
of discordance between the gene tree and the population tree using some
basic ideas from coalescent theory. That leads to a further refinement,
using coalescent theory directly to examine alternative biogeographic
hypotheses.</p>
<h1 class="unnumbered"
id="coalescent-based-estimates-of-migration-rate">Coalescent-based
estimates of migration rate</h1>
<p>Peter Beerli and Joe Felsenstein <span class="citation"
data-cites="Beerli-Felsenstein-1999 Beerli-Felsenstein-2001"></span>
proposed a coalescent-based method to estimate migration rates among
populations. As with other analytical methods we’ve encountered in this
course, the details can get pretty hairy, but the basic idea is
(relatively) simple.</p>
<p>Recall that in a single population we can describe the coalescent
history of a sample without too much difficulty. Specifically, given a
sample of <span class="math inline">\(k\)</span> alleles in a diploid
population with effective size <span class="math inline">\(N_e\)</span>,
the probability that the first coalescent event took place <span
class="math inline">\(t\)</span> generations ago is <span
class="math display">\[P(t|k, N_e) =
\left(\frac{k(k-1)}{4N_e}\right)\left(1-
  \frac{k(k-1)}{4N_e}\right)^{t-1} \quad .
\label{eq:coalescent-single}\]</span> Now suppose that we have a sample
of alleles from <span class="math inline">\(K\)</span> different
populations. To keep things (relatively) simple, we’ll imagine that we
have a sample of <span class="math inline">\(n\)</span> alleles from
every one of these populations and that every population has an
effective size of <span class="math inline">\(N_e\)</span>. In addition,
we’ll imagine that there is migration among populations, but again we’ll
keep it really simple. Specifically, we’ll assume that the probability
that a given allele in our sample from one population had its ancestor
in a different population in the immediately preceding generation is
<span class="math inline">\(m\)</span>.<a href="#fn1"
class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a>
Under this simple scenario, we can again construct the coalescent
history of our sample. How? Funny you should ask.</p>
<p>We start by using the same logic we used to construct equation (<a
href="#eq:coalescent-single" data-reference-type="ref"
data-reference="eq:coalescent-single">[eq:coalescent-single]</a>).
Specifically, we ask “What’s the probability of an ‘event’ in the
immediately preceding generation?” The complication is that there are
two kinds of events possible:</p>
<ol>
<li><p>a coalescent event and</p></li>
<li><p>a migration event.</p></li>
</ol>
<p>As in our original development of the coalescent process, we’ll
assume that the population sizes are large enough that the probability
of two coalescent events in a single time step is so small as to be
negligible. In addition, we’ll assume that the number of populations and
the migration rates are small enough that the probability of more than
one event of either type is so small as to be negligible. That means
that all we have to do is to calculate the probability of either a
coalescent event or a migration event and combine them to calculate the
probability of an event. It turns out that it’s easiest to calculate the
probability that there <span> <em>isn’t</em></span> an event first and
then to calculate the probability that there is an event as one minus
that.</p>
<p>We already know that the probability of a coalescent event in
population <span class="math inline">\(k\)</span>, is <span
class="math display">\[P_k(\mbox{coalescent}|n, N_e) =
\frac{k(k-1)}{4N_e} \quad ,\]</span> so the probability that there is
<span><em>not</em></span> a coalescent event in any of our <span
class="math inline">\(K\)</span> populations is <span
class="math display">\[P(\mbox{no coalescent}|k, N_e, K) =
\left(1-\frac{k(k-1)}{4N_e}\right)^K
\quad .\]</span> If <span class="math inline">\(m\)</span> is the
probability that there was a migration event in a particular population
than the probability that there is <span><em>not</em></span> a migration
event involving any of our <span class="math inline">\(kK\)</span>
alleles<a href="#fn2" class="footnote-ref" id="fnref2"
role="doc-noteref"><sup>2</sup></a> is <span
class="math display">\[P(\mbox{no migration}|k,m, K) = (1-m)^{kK} \quad
.\]</span> So the probability that there <span><em>is</em></span> an
event of some kind is <span class="math display">\[P(\mbox{event}|k, m,
N_e, K) = 1 - P(\mbox{no coalescent}|k, N_e,
K)P(\mbox{no migration}|k,m, K) \quad .\label{eq:event}\]</span> Now we
can calculate the time back to the first event <span
class="math display">\[P(\mbox{event at }t|k, m, N_e, K) =
P(\mbox{event}|k, m, N_e,
K)\left(1 - P(\mbox{event}|k, m, N_e, K)\right)^{t-1} \quad .
\label{eq:time-to-event}\]</span> We can then use Bayes theorem to
calculate the probability that the event was a coalescence or a
migration and the population or populations involved. Notice, however,
that if the event is a coalescent event, we first have to pick the
population in which it occurred and then identify the pair of alleles
that coalesced. Alleles have to be in the same population. Once we’ve
done all of this, we have a new population configuration and we can
start over. We continue until all of the alleles have coalesced into a
single common ancestor, and then we have the complete coalescent history
of our sample.<a href="#fn3" class="footnote-ref" id="fnref3"
role="doc-noteref"><sup>3</sup></a> That’s roughly the logic that Beerli
and Felsenstein use to construct coalescent histories for a sample of
alleles from a set of populationsexcept that they allow effective
population sizes to differ among populations and they allow migration
rates to differ among all pairs of populations. As if that weren’t bad
enough, now things start to get even more complicated.</p>
<p>There are lots of different coalescent histories possible for a
sample consisting of <span class="math inline">\(n\)</span> alleles from
each of <span class="math inline">\(K\)</span> different populations,
even when we fix <span class="math inline">\(m\)</span> and <span
class="math inline">\(N_e\)</span>. Worse yet, given any one coalescent
history, there are a lot of different possible mutational histories
possible. In short, there are a lot of different possible sample
configurations consistent with a given set of migration rates and
effective population size. Nonetheless, some combinations of <span
class="math inline">\(m\)</span> and <span
class="math inline">\(N_e\)</span> will make the data more likely than
others. In other words, we can construct a likelihood for our data:
<span class="math display">\[P(\mbox{data}|m, N_e) \propto f(n, m, N_e,
K) \quad ,\]</span> where <span class="math inline">\(f(n, m,
N_e,K)\)</span> is some very complicated function of the probabilities
we derived above. In fact, the function is so complicated, we can’t even
write it down. Fortunately, Beerli and Felsenstein, being very clever
people, figured out a way to simulate the likelihood, and
<span><code>Migrate-n</code></span> <a
href="http://popgen.sc.fsu.edu/Migrate/Migrate-n.html"
class="uri">http://popgen.sc.fsu.edu/Migrate/Migrate-n.html</a> provides
a (relatively) simple way that you can use your data to estimate <span
class="math inline">\(m\)</span> and <span
class="math inline">\(N_e\)</span> for a set of populations. In fact,
<span><code>Migrate-N</code></span> will allow you to estimate pairwise
migration rates among all populations in your sample, and since it can
simulate a likelihood, if you put priors on the parameters you’re
interested in, i.e., <span class="math inline">\(m\)</span> and <span
class="math inline">\(N_e\)</span>, you can get Bayesian estimates of
those parameters rather than maximum likelihood estimates, including
credible intervals around those estimates so that you have a good sense
of how reliable your estimates are.<a href="#fn4" class="footnote-ref"
id="fnref4" role="doc-noteref"><sup>4</sup></a></p>
<p>There’s one further complication I need to mention, and it involves a
lie I just told you. <span><code>Migrate-N</code></span> can’t give you
estimates of <span class="math inline">\(m\)</span> and <span
class="math inline">\(N_e\)</span>. Remember how every time we’ve dealt
with drift and another process we always end up with things like <span
class="math inline">\(4N_em\)</span>, <span
class="math inline">\(4N_e\mu\)</span>, and the like. Well, the
situation is no different here. What <span><code>Migrate-N</code></span>
can actually estimate are the two parameters <span
class="math inline">\(4N_em\)</span> and <span
class="math inline">\(\theta=4N_e\mu\)</span>.<a href="#fn5"
class="footnote-ref" id="fnref5" role="doc-noteref"><sup>5</sup></a> How
did <span class="math inline">\(\mu\)</span> get in here when I only
mentioned it in passing? Well, remember that I said that once the
computer has constructed a coalescent history, it has to apply mutations
to that history. Without mutation, all of the alleles in our sample
would be identical to one another. Mutation is what produces the
diversity. So what we get from <span><code>Migrate-N</code></span> isn’t
the fraction of a population that’s composed of migrants. Rather, we get
an estimate of how much migration contributes to local population
diversity relative to mutation. That’s a pretty interesting estimate to
have, but it may not be everything that we want.</p>
<p>There’s a further complication to be aware of. Think about the
simulation process I described. All of the alleles in our sample are
descended from a single common ancestor. That means we are implicitly
assuming that the set of populations we’re studying have been around
long enough and have been exchanging migrants with one another long
enough that we’ve reached a drift-mutation-migration equilibrium. If
we’re dealing with a relatively small number of populations in a
geographically limited area, that may not be an unreasonable assumption,
but what if we’re dealing with populations of crickets spread across all
of the northern Rocky Mountains? And what if we haven’t sampled all of
the populations that exist?<a href="#fn6" class="footnote-ref"
id="fnref6" role="doc-noteref"><sup>6</sup></a> In many circumstances,
it may be more appropriate to imagine that populations diverged from one
another at some time in the not too distant past, have exchanged genes
since their divergence, but haven’t had time to reach a
drift-mutation-migration equilibrium. What do we do then?</p>
<h1 class="unnumbered" id="divergence-and-migration">Divergence and
migration</h1>
<p>Rasmus Nielsen and John Wakely <span class="citation"
data-cites="Nielsen-Wakeley-2001"></span> consider the simplest
generalization of Beerli and Felsenstein <span class="citation"
data-cites="Beerli-Felsenstein-1999 Beerli-Felsenstein-2001"></span> you
could imagine (Figure <a href="#fig:nielsen-wakeley"
data-reference-type="ref" data-reference="fig:nielsen-wakeley">2</a>).
They consider a situation in which you have samples from only two
populations and you’re interested in determining both how long ago the
populations diverged from one another and how much gene exchange there
has been between the populations since they diverged. As in
<span><code>Migrate-N</code></span> mutation and migration rates are
confounded with effective population size, and the relevant parameters
become:</p>
<ul>
<li><p><span class="math inline">\(\theta_a\)</span>, which is <span
class="math inline">\(4N_e\mu\)</span> in the ancestral
population.</p></li>
<li><p><span class="math inline">\(\theta_1\)</span>, which is <span
class="math inline">\(4N_e\mu\)</span> in the first population.</p></li>
<li><p><span class="math inline">\(\theta_2\)</span>, which is <span
class="math inline">\(4N_e\mu\)</span> in the second
population.</p></li>
<li><p><span class="math inline">\(M_1\)</span>, which is <span
class="math inline">\(2N_em_1\)</span> in the first population, where
<span class="math inline">\(m_1\)</span> is the fraction of the first
population composed of migrants from the second population.</p></li>
<li><p><span class="math inline">\(M_2\)</span>, which is <span
class="math inline">\(2N_em_2\)</span> in the second
population.</p></li>
<li><p><span class="math inline">\(T\)</span>, which is the time since
the populations diverged. Specifically, if there have been <span
class="math inline">\(t\)</span> units since the two populations
diverged, <span class="math inline">\(T=t/2N_1\)</span>, where <span
class="math inline">\(N_1\)</span> is the effective size of the first
population.</p></li>
</ul>
<figure id="fig:nielsen-wakeley">
<div class="center">
<img src="nielsen-wakeley.png" style="height:6cm" />
</div>
<figcaption>The simple model developed by Nielsen and Wakeley <span
class="citation" data-cites="Nielsen-Wakeley-2001"></span>. <span
class="math inline">\(\theta_a\)</span> is <span
class="math inline">\(4N_e\mu\)</span> in the ancestral population;
<span class="math inline">\(\theta_1\)</span> and <span
class="math inline">\(\theta_2\)</span> are <span
class="math inline">\(4N_e\mu\)</span> in the descendant populations;
<span class="math inline">\(M_1\)</span> and <span
class="math inline">\(M_2\)</span> are <span
class="math inline">\(2N_em\)</span>, where <span
class="math inline">\(m\)</span> is the backward migration rate; and
<span class="math inline">\(T\)</span> is the time since divergence of
the two populations.</figcaption>
</figure>
<p>Given that set of parameters, you can probably imagine that you can
calculate the likelihood of the data for a given set of parameters.<a
href="#fn7" class="footnote-ref" id="fnref7"
role="doc-noteref"><sup>7</sup></a> Once you can do that you can either
obtain maximum-likelihood estimates of the parameters by maximizing the
likelihood, or you can place prior distributions on the parameters and
obtain Bayesian estimates from the posterior distribution. Either way,
armed with estimates of <span class="math inline">\(\theta_a\)</span>,
<span class="math inline">\(\theta_1\)</span>, <span
class="math inline">\(\theta_2\)</span>, <span
class="math inline">\(M_1\)</span>, <span
class="math inline">\(M_2\)</span>, and <span
class="math inline">\(T\)</span> you can say something about:</p>
<ol>
<li><p>the effective population sizes of the two populations relative to
one another and relative to the ancestral population,</p></li>
<li><p>the relative frequency with which migrants enter each of the two
populations from the other, and</p></li>
<li><p>the time at which the two populations diverged from one
another.</p></li>
</ol>
<p>Keep in mind, though, that the estimates of <span
class="math inline">\(M_1\)</span> and <span
class="math inline">\(M_2\)</span> confound local effective population
sizes with migration rates. So if <span class="math inline">\(M_1 &gt;
M_2\)</span>, for example, it does not mean that the fraction of
migrants incorporated into population 1 exceeds the fraction
incorporated into population 2. It means that the
<span><em>number</em></span> of migrants entering population 1 is
greater than the number entering population 2.</p>
<h2 class="unnumbered" id="an-example">An example</h2>
<p>Orti et al. <span class="citation"
data-cites="Orti-etal-1994"></span> report the results of phylogenetic
analyses of mtDNA sequences from 25 populations of threespine
stickleback, <span><em>Gasterosteus aculeatus</em></span>, in Europe,
North America, and Japan. The data consist of sequences from a 747bp
fragment of cytochrome <span class="math inline">\(b\)</span>. Nielsen
and Wakely <span class="citation"
data-cites="Nielsen-Wakeley-2001"></span> analyze these data using their
approach. Their analyses show that “[a] model of moderate migration and
very long divergence times is more compatible with the data than a model
of short divergence times and low migration rates.” By “very long
divergence times” they mean <span class="math inline">\(T
&gt; 4.5\)</span>, i.e., <span class="math inline">\(t &gt;
4.5N_1\)</span>. Focusing on populations in the western (population 1)
and eastern Pacific (population 2), they find that the maximum
likelihood estimate of <span class="math inline">\(M_1\)</span> is 0,
indicating that there is little if any gene flow from the eastern
Pacific (population 2) into the western Pacific (population 1). In
contrast, the maximum likelihood estimate of <span
class="math inline">\(M_2\)</span> is about 0.5, indicating that one
individual is incorporated into the eastern Pacific population from the
western Pacific population every other generation. The
maximum-likelihood estimates of <span
class="math inline">\(\theta_1\)</span> and <span
class="math inline">\(\theta_2\)</span> indicate that the effective size
of the population eastern Pacific population is about 3.0 times greater
than that of the western Pacific population.</p>
<h2 class="unnumbered"
id="extending-the-approach-to-multiple-populations">Extending the
approach to multiple populations</h2>
<p>Jody Hey later announced the release of
<span><code>IMa2</code></span>.<a href="#fn8" class="footnote-ref"
id="fnref8" role="doc-noteref"><sup>8</sup></a> Building on work
described in Hey and Nielsen <span class="citation"
data-cites="Hey-Nielsen-2004 Hey-Nielsen-2007"></span>,
<span><code>IMa2</code></span> allows you to estimate relative
divergence times, relative effective population sizes, and relative
pairwise migration rates for more than two populations at a time. That
flexibility comes at a cost, of course. In particular, you have to
specify the phylogenetic history of the populations before you begin the
analysis.</p>
<h1 class="unnumbered"
id="phylogeography-of-montane-grasshoppers">Phylogeography of montane
grasshoppers</h1>
<p>Lacey Knowles studied grasshoppers in the genus
<span><em>Melanopus</em></span>. She collected 1275bp of DNA sequence
data from cytochrome oxidase I (COI) from 124 individuals of
<span><em>M. oregonensis</em></span> and two outgroup species. The
specimens were collected from 15 “sky-island” sites in the northern
Rocky Mountains (see Figure <a href="#fig:sky-islands"
data-reference-type="ref" data-reference="fig:sky-islands">3</a>; <span
class="citation" data-cites="Knowles-2001"></span>). Two alternative
hypotheses had been proposed to describe the evolutionary relationships
among these grasshoppers (refer to Figure <a
href="#fig:divergence-hypotheses" data-reference-type="ref"
data-reference="fig:divergence-hypotheses">4</a> for a pictorial
representation):</p>
<figure id="fig:sky-islands">
<div class="center">
<img src="sky-islands.png" style="height:6cm" />
</div>
<figcaption>Collection sites for <span><em>Melanopus
oregonensis</em></span> in the northern Rocky Mountains (from <span
class="citation" data-cites="Knowles-2001"></span>).</figcaption>
</figure>
<ul>
<li><p><span><strong>Widespread ancestor</strong></span>: The existing
populations might represent independently derived remnants of a single,
widespread population. In this case all of the populations would be
equally related to one another.</p></li>
<li><p><span><strong>Multiple glacial refugia</strong></span>:
Populations that shared the same refugium will be closely related while
those that were in different refugia will be distantly related.</p></li>
</ul>
<figure id="fig:divergence-hypotheses">
<div class="center">
<img src="divergence-hypotheses.png" style="height:6cm" />
</div>
<figcaption>Pictorial representations of the “widespread ancestor” (top)
and “glacial refugia” (bottom) hypotheses (from <span class="citation"
data-cites="Knowles-2001"></span>).</figcaption>
</figure>
<p>As is evident from Figure <a href="#fig:divergence-hypotheses"
data-reference-type="ref"
data-reference="fig:divergence-hypotheses">4</a>, the two hypotheses
have very different consequences for the coalescent history of alleles
in the sample. Since the interrelationships between divergence times and
time to common ancestry differ so markedly between the two scenarios,
the pattern of sequence differences found in relation to the geographic
distribution will differ greatly between the two scenarios.</p>
<p>Using techniques described in Knowles and Maddison <span
class="citation" data-cites="Knowles-Maddison-2002"></span>, Knowles
simulated gene trees under the widespread ancestor hypothesis. She then
placed them within a population tree representing the multiple glacial
refugia hypothesis and calculated a statistic, <span
class="math inline">\(s\)</span>, that measures the discordance between
a gene tree and the population tree that contains it. This gave her a
distribution of <span class="math inline">\(s\)</span> under the
widespread ancestor hypothesis. She compared the <span
class="math inline">\(s\)</span> estimated from her actual data with
this distribution and found that the observed value of <span
class="math inline">\(s\)</span> was only 1/2 to 1/3 the size of the
value observed in her simulations.<a href="#fn9" class="footnote-ref"
id="fnref9" role="doc-noteref"><sup>9</sup></a> Let’s unpack that a
bit.</p>
<ul>
<li><p>Knowles estimated the phylogeny of the haplotypes in her sample.
<span class="math inline">\(s\)</span> is the estimated minimum number
of among-population migration events necessary to account for where
haplotypes are currently found given the inferred phylogeny <span
class="citation" data-cites="Slatkin-Maddison-1989"></span>. Let’s call
the <span class="math inline">\(s\)</span> estimated from the data <span
class="math inline">\(s_{obs}\)</span>.</p></li>
<li><p>Then she simulated a neutral coalescence process in which the
populations were derived from a single, widespread ancestral population.
For each simulation she rearranged the data so that populations were
grouped into separate refugia and estimated <span
class="math inline">\(s_{sim}\)</span> from the rearranged data, and she
repeated this 100 times for several different times since population
splitting.</p></li>
</ul>
<p>The results are shown in Figure <a href="#fig:knowles-s-values"
data-reference-type="ref" data-reference="fig:knowles-s-values">5</a>.
As you can see, the observed <span class="math inline">\(s\)</span>
value is much smaller than any of those obtained from the coalescent
simulations. That means that the observed data require far fewer
among-population migration events to account for the observed geographic
distribution of haplotypes than would be expected with independent
origin of the populations from a single, widespread ancestor. In short,
Knowles presented strong evidence that her data are not consistent with
the widespread ancestor hypothesis.</p>
<figure id="fig:knowles-s-values">
<div class="center">
<img src="knowles-s-values.png" style="height:10cm" />
</div>
<figcaption>Distribution of the observed minimum number of
among-population migration events, <span
class="math inline">\(s\)</span>, and the expected minimum number of
migration events under the “widespread ancestor” hypothesis. (from <span
class="citation" data-cites="Knowles-2001"></span>).</figcaption>
</figure>
<h1 class="unnumbered"
id="approximate-bayesian-computation-motivation">Approximate Bayesian
computation: motivation</h1>
<p>Approximate Bayesian Computation (ABC for short), extends the basic
idea Knowles used to consider more complicated scenarios. The
<span><code> IMa</code></span> approach developed by Nielsen, Wakely,
and Hey is potentially <span><em>very</em></span> flexible and
<span><em>very</em></span> powerful <span class="citation"
data-cites="Hey-Nielsen-2004 Hey-Nielsen-2007 Nielsen-Wakeley-2001"></span>.
It allows for non-equilibrium scenarios in which the populations from
which we sampled diverged from one another at different times, but
suppose that we think our populations have dramatically increased in
size over time (as in humans) or dramatically changed their distribution
(as with an invasive species). Is there a way to use genetic data to
gain some insight into those processes? Would I be asking that question
if the answer were “No”?</p>
<h1 class="unnumbered" id="an-example-1">An example</h1>
<p>Let’s change things up a bit this time and start with an example of a
problem we’d like to solve first. Once you see what the problem is, then
we can talk about how we might go about solving it. The case we’ll
discuss is the case of the cane toad, <span><em>Bufo
marinus</em></span>, in Australia.</p>
<p>You may know that the cane toad is native to the American tropics. It
was purposely introduced into Australia in 1935 as a biocontrol agent,
where it has spread across an area of more than 1 million km<span
class="math inline">\(^2\)</span>. Its range is still expanding in
northern Australia and to a lesser extent in eastern
Australia (Figure <a href="#fig:cane-toad-expansion"
data-reference-type="ref"
data-reference="fig:cane-toad-expansion">6</a>).<a href="#fn10"
class="footnote-ref" id="fnref10" role="doc-noteref"><sup>10</sup></a>
Estoup et al. <span class="citation"
data-cites="Estoup-etal-2004"></span> collected microsatellite data from
30 individuals in each of 19 populations along roughly linear transects
in the northern and eastern expansion areas.</p>
<figure id="fig:cane-toad-expansion">
<div class="center">
<img src="cane-toad-expansion.png" style="width:6in" />
</div>
<figcaption>Maps showing the expansion of the cane toad population in
Australia since its introduction in 1935 (from <span class="citation"
data-cites="Estoup-etal-2004"></span>).</figcaption>
</figure>
<p>With these data they wanted to distinguish among five possible
scenarios describing the geographic spread:</p>
<ul>
<li><p><span><strong>Isolation by distance</strong></span>: As the
expansion proceeds, each new population is founded by or immigrated into
by individuals with a probability proportional to the distance from
existing populations.</p></li>
<li><p><span><strong>Differential migration and
founding</strong></span>: Identical to the preceding model except that
the probability of founding a population may be different from the
probability of immigration into an existing population.</p></li>
<li><p><span><strong>“Island” migration and founding</strong></span>:
New populations are established from existing populations without
respect to the geographic distances involved, and migration occurs among
populations without respect to the distances involved.</p></li>
<li><p><span><strong>Stepwise migration and founding with founder
events</strong></span>: Both migration and founding of populations
occurs only among immediately adjacent populations. Moreover, when a new
population is established, the number of individuals involved may be
very small.</p></li>
<li><p><span><strong>Stepwise migration and founding without founder
events</strong></span>: Identical to the preceding model except that
when a population is founded its size is assumed to be equal to the
effective population size.</p></li>
</ul>
<p>That’s a pretty complex set of scenarios. Clearly, you could use
<span><code> Migrate</code></span> or <span><code>IMa2</code></span> to
estimate parameters from the data Estoup et al. <span class="citation"
data-cites="Estoup-etal-2004"></span> report, but would those parameters
allow you to distinguish those scenarios? Not in any straightforward way
that I can see. Neither <span><code>Migrate</code></span> nor
<span><code>IMa2</code></span> distinguishes between founding and
migration events for example. And with <span><code>IMa2</code></span>
we’d have to specify the relationships among our sampled populations
before we could make any of the calculations. In this case we want to
test alternative hypotheses of population relationship. So what do we
do?</p>
<h1 class="unnumbered" id="approximate-bayesian-computation">Approximate
Bayesian Computation</h1>
<p>Well, in principle we could take an approach similar to what
<span><code> Migrate</code></span> and <span><code>IMa2</code></span>
use. Let’s start by reviewing what we did last time<a href="#fn11"
class="footnote-ref" id="fnref11" role="doc-noteref"><sup>11</sup></a>
with <span><code>Migrate</code></span> and
<span><code>IMa2</code></span>. In both cases, we knew how to simulate
data given a set of mutation rates, migration rates, local effective
population sizes, and times since divergence. Let’s call that whole,
long string of parameters <span class="math inline">\(\xi\)</span> and
our big, complicated data set <span class="math inline">\(X\)</span>. If
we run enough simulations, we can keep track of how many of those
simulations produce data identical to the data we collected. With those
results in hand, we can estimate <span
class="math inline">\(P(X|\xi)\)</span>, the likelihood of the data, as
the fraction of simulations that produce data identical to the data we
collected.<a href="#fn12" class="footnote-ref" id="fnref12"
role="doc-noteref"><sup>12</sup></a> In principle, we could take the
same approach in this, much more complicated, situation. But the problem
is that there are an astronomically large number of different possible
coalescent histories and different allelic configurations possible with
any one population history both because the population histories being
considered are pretty complicated and because the coalescent history of
every locus will be somewhat different from the coalescent history at
other loci. As a result, the chances of getting
<span><em>any</em></span> simulated samples that match our actual
samples is virtually nil, and we can’t estimate <span
class="math inline">\(P(X|\xi)\)</span> in the way we have so far.</p>
<p>Approximate Bayesian computation is an approach that allows us to get
around this problem. It was introduced by Beaumont et al. <span
class="citation" data-cites="Beaumont-etal-2002"></span> precisely to
allow investigators to get approximate estimates of parameters and data
likelihoods in a Bayesian framework. Again, the details of the
implementation get pretty hairy,<a href="#fn13" class="footnote-ref"
id="fnref13" role="doc-noteref"><sup>13</sup></a> but the basic idea is
relatively straightforward.<a href="#fn14" class="footnote-ref"
id="fnref14" role="doc-noteref"><sup>14</sup></a></p>
<ol>
<li><p>Calculate “appropriate” summary statistics for your data set,
e.g., pairwise estimates of <span
class="math inline">\(\phi_{ST}\)</span> (possibly one for every locus
if you’re using microsatellite or SNP data), estimates of within
population diversity, counts of the number of segregating sites (for
nucleotide sequence data, both within each population and across the
entire sample) or counts of the number of segregating alleles (for
microsatellite data). Call that set of summary statistics <span
class="math inline">\(S\)</span>.</p></li>
<li><p>Specify a prior distribution for the unknown parameters, <span
class="math inline">\(\xi\)</span>.</p></li>
<li><p>Pick a random set of parameter values, <span
class="math inline">\(\xi&#39;\)</span> from the prior distribution and
simulate a data set for that set of parameter values.</p></li>
<li><p>Calculate the same summary statistics for the simulated data set
as you calculated for your actual data. Call that set of statistics
<span class="math inline">\(S&#39;\)</span>.</p></li>
<li><p>Calculate the distance between <span
class="math inline">\(S\)</span> and <span
class="math inline">\(S&#39;\)</span>.<a href="#fn15"
class="footnote-ref" id="fnref15" role="doc-noteref"><sup>15</sup></a>
Call it <span class="math inline">\(\delta\)</span>. If it’s less than
some value you’ve decided on, <span
class="math inline">\(\delta^*\)</span>, keep track of <span
class="math inline">\(S&#39;\)</span> and the associated <span
class="math inline">\(\xi&#39;\)</span> and <span
class="math inline">\(\delta\)</span>. Otherwise, throw all of them away
and forget you ever saw them.</p></li>
<li><p>Return to step 2 and repeat until you have accepted a large
number of pairs of <span class="math inline">\(S&#39;\)</span> and <span
class="math inline">\(\xi&#39;\)</span>.</p></li>
</ol>
<p>Now you have a bunch of <span class="math inline">\(S&#39;\)</span>s
and a bunch of <span class="math inline">\(\xi&#39;\)</span>s that
produced them. Let’s label them <span class="math inline">\(S_i\)</span>
and <span class="math inline">\(\xi_i\)</span>, and let’s remember what
we’re trying to do. We’re trying to estimate <span
class="math inline">\(\xi\)</span> for our real data. What we have from
our real data is <span class="math inline">\(S\)</span>. So far it seems
as if we’ve worked our computer pretty hard, but we haven’t made any
progress.</p>
<p>Here’s where the trick comes in. Suppose we fit a regression to the
data we’ve simulated <span class="math display">\[\xi_i = \alpha +
S_i\beta + \epsilon \quad ,\]</span> where <span
class="math inline">\(\alpha\)</span> is an intercept, <span
class="math inline">\(\beta\)</span> is a vector of regression
coefficients relating each of the summary statistics to <span
class="math inline">\(\xi\)</span>, and <span
class="math inline">\(\epsilon\)</span> is an error vector.<a
href="#fn16" class="footnote-ref" id="fnref16"
role="doc-noteref"><sup>16</sup></a> Once we’ve fit this regression, we
can use it to predict what <span class="math inline">\(\xi\)</span>
should be in our real data, namely <span class="math display">\[\xi =
\alpha + S\beta \quad ,\]</span> where the <span
class="math inline">\(S\)</span> here corresponds to our observed set of
summary statistics. If we throw in some additional bells and whistles,
we can approximate the posterior distribution of our parameters. With
that we can get not only a point estimate for <span
class="math inline">\(\xi\)</span>, but also credible intervals for all
of its components.</p>
<h1 class="unnumbered" id="back-to-the-real-world">Back to the real
world<a href="#fn17" class="footnote-ref" id="fnref17"
role="doc-noteref"><sup>17</sup></a></h1>
<p>OK. So now we know how to do ABC, how do we apply it to the cane toad
data. Well, using the additional bells and whistles I mentioned, we end
up with a whole distribution of <span
class="math inline">\(\delta\)</span> for each of the scenarios we try.
The scenario with the smallest <span
class="math inline">\(\delta\)</span> provides the best fit of the model
to the data. In this case, that corresponds to model 4, the stepwise
migration with founder model, although it is only marginally better than
model 1 (isolation by distance) and model 2 (isolation by distance with
differential migration and founding) in the northern expansion
area (Figure <a href="#fig:cane-toad-models" data-reference-type="ref"
data-reference="fig:cane-toad-models">7</a>).</p>
<figure id="fig:cane-toad-models">
<div class="center">
<img src="cane-toad-models.png" style="width:4.5in" />
</div>
<figcaption>Posterior distribution of <span
class="math inline">\(\delta\)</span> for the five models considered in
Estoup et al. <span class="citation"
data-cites="Estoup-etal-2004"></span>.</figcaption>
</figure>
<p>Of course, we also have estimates for various parameters associated
with this model:</p>
<ul>
<li><p><span class="math inline">\(N_{e_s}\)</span>: the effective
population size when the population is stable.</p></li>
<li><p><span class="math inline">\(N_{e_f}\)</span>: the effective
population size when a new population is founded.</p></li>
<li><p><span class="math inline">\(F_R\)</span>: the founding ratio,
<span class="math inline">\(N_{e_s}/N_{e_f}\)</span>.</p></li>
<li><p><span class="math inline">\(m\)</span>: the migration
rate.</p></li>
<li><p><span class="math inline">\(N_{e_s}m\)</span>: the effective
number of migrants per generation.</p></li>
</ul>
<p>The estimates are summarized in Table <a href="#table:cane-toad"
data-reference-type="ref" data-reference="table:cane-toad">1</a>.
Although the credible intervals are fairly broad,<a href="#fn18"
class="footnote-ref" id="fnref18" role="doc-noteref"><sup>18</sup></a>
there are a few striking features that emerge from this analysis.</p>
<ul>
<li><p>Populations in the northern expansion area are larger, than those
in the eastern expansion region. Estoup et al. <span class="citation"
data-cites="Estoup-etal-2004"></span> suggest that this is consistent
with other evidence suggesting that ecological conditions are more
homogeneous in space and more favorable to cane toads in the north than
in the east.</p></li>
<li><p>A smaller number of individuals is responsible for founding new
populations in the east than in the north, and the ratio of
“equilibrium” effective size to the size of the founding population is
bigger in the east than in the north. (The second assertion is only
weakly supported by the results.)</p></li>
<li><p>Migration among populations is more limited in the east than in
the north.</p></li>
</ul>
<p>As Estoup et al. <span class="citation"
data-cites="Estoup-etal-2004"></span> suggest, results like these could
be used to motivate and calibrate models designed to predict the future
course of the invasion, incorporating a balance between gene flow (which
can reduce local adaptation), natural selection, drift, and colonization
of new areas.</p>
<div class="center">
<div id="table:cane-toad">
<table>
<caption>Posterior means and 90% credible intervals for parameters of
model 4 in the eastern and northern expansion areas of <span><em>Bufo
marinus</em></span>.</caption>
<thead>
<tr class="header">
<th style="text-align: center;">Parameter</th>
<th style="text-align: center;">area</th>
<th style="text-align: right;">mean (5%, 90%)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><span
class="math inline">\(N_{e_s}\)</span></td>
<td style="text-align: center;">east</td>
<td style="text-align: right;">744 (205, 1442)</td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">north</td>
<td style="text-align: right;">1685 (526, 2838)</td>
</tr>
<tr class="odd">
<td style="text-align: center;"><span
class="math inline">\(N_{e_f}\)</span></td>
<td style="text-align: center;">east</td>
<td style="text-align: right;">78 (48, 118)</td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">north</td>
<td style="text-align: right;">311 (182, 448)</td>
</tr>
<tr class="odd">
<td style="text-align: center;"><span
class="math inline">\(F_R\)</span></td>
<td style="text-align: center;">east</td>
<td style="text-align: right;">10.7 (2.4, 23.8)</td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">north</td>
<td style="text-align: right;">5.9 (1.6, 11.8)</td>
</tr>
<tr class="odd">
<td style="text-align: center;"><span
class="math inline">\(m\)</span></td>
<td style="text-align: center;">east</td>
<td style="text-align: right;">0.014 (<span class="math inline">\(6.0
\times 10^{-6}\)</span>, 0.064)</td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">north</td>
<td style="text-align: right;">0.117 (<span class="math inline">\(1.4
\times 10^{-4}\)</span>, 0.664)</td>
</tr>
<tr class="odd">
<td style="text-align: center;"><span
class="math inline">\(N_{e_s}m\)</span></td>
<td style="text-align: center;">east</td>
<td style="text-align: right;">4.7 (0.005, 19.9)</td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">north</td>
<td style="text-align: right;">188 (0.023, 883)</td>
</tr>
</tbody>
</table>
</div>
</div>
<h1 class="unnumbered" id="limitations-of-abc">Limitations of ABC</h1>
<p>If you’ve learned anything by now, you should have learned that there
is no perfect method. An obvious disadvantage of ABC relative to either
<span><code>Migrate</code></span> or <span><code>IMa2</code></span> is
that it is much more computationally intensive.</p>
<ul>
<li><p>Because the scenarios that can be considered are much more
complex, it simply takes a long time to simulate all of the
data.</p></li>
<li><p>In the last few years, one of the other disadvantagesthat you had
to know how to do some moderately complicated scripting to piece
together several different packages in order to run analysishas become
less of a problem. <span><code>popABC</code></span> (<a
href="http://code.google.com/p/popabc/"
class="uri">http://code.google.com/p/popabc/</a>,
<span><code>DIYABC</code></span> (<a
href="http://www1.montpellier.inra.fr/CBGP/diyabc/"
class="uri">http://www1.montpellier.inra.fr/CBGP/diyabc/</a>), and the
<span><code> abc</code></span> library in <span><code>R</code></span>
make it <span><em>relatively</em></span> easy<a href="#fn19"
class="footnote-ref" id="fnref19" role="doc-noteref"><sup>19</sup></a>
to perform the simulations.</p></li>
<li><p>Selecting an appropriate set of summary statistics isn’t easy,
and it turns out that which set is most appropriate may depend on the
value of the parameters that you’re trying to estimate and the which of
the scenarios that you’re trying to compare is closest to the actual
scenario applying to the populations from which you collected the data.
Of course, if you knew what the parameter values were and which scenario
was closest to the actual scenario, you wouldn’t need to do ABC in the
first place.</p></li>
<li><p>In the end, ABC allows you to compare a small number of
evolutionary scenarios. It can tell you which of the scenarios you’ve
imagined provides the best combination of fit to the data and
parsimonious use of parameters (if you choose model comparison
statistics that include both components), but it takes additional work
to determine whether the model is adequate, in the sense that it does a
good job of explaining the data. Moreover, even if you determine that
the model is adequate, you can’t exclude the possibility that there are
other scenarios that might be equally adequateor even better.</p></li>
</ul>
<h1 class="unnumbered" id="creative-commons-license">Creative Commons
License</h1>
<p>These notes are licensed under the Creative Commons Attribution
License. To view a copy of this license, visit or send a letter to
Creative Commons, 559 Nathan Abbott Way, Stanford, California 94305,
USA.</p>
<aside id="footnotes" class="footnotes footnotes-end-of-document"
role="doc-endnotes">
<hr />
<ol>
<li id="fn1"><p>In other words, <span class="math inline">\(m\)</span>
is the backwards migration rate, the probability that a gene in one
population came from another population in the preceding generation.
This is the same migration rate we encountered weeks ago when we
discussed the balance between drift and migration. The method Beerli and
Felsenstein developed allows populations to differ in <span
class="math inline">\(N_e\)</span> and allows rates of migration among
pairs of populations to differ. It even allows the rate of migration
into population A from population B to differ from the rate of migration
into population B from population A. We’re going to ignore all of those
complications here, because the math is complicated enough without them,
and it gets a <span><em>lot</em></span> more complicated when they are
included.<a href="#fnref1" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn2"><p><span class="math inline">\(K\)</span> populations each
with <span class="math inline">\(k\)</span> alleles<a href="#fnref2"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn3"><p>This may not seem very simple, but just think about how
complicated it would be if I allowed every population to have a
different effective size and if I allowed each pair of populations to
have different migration rates between them.<a href="#fnref3"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn4"><p>If you’d like to see a comparision of maximum likelihood
and Bayesian approaches, Beerli <span class="citation"
data-cites="Beerli-2006"></span> provides an excellent overview.<a
href="#fnref4" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn5"><p>Depending on the option you pick when you run
<span><code>Migrate</code></span> you can either get <span
class="math inline">\(\theta\)</span> and <span
class="math inline">\(4N_em\)</span> or <span
class="math inline">\(\theta\)</span> and <span
class="math inline">\(M=m/\mu\)</span>.<a href="#fnref5"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn6"><p>Beerli <span class="citation"
data-cites="Beerli-2004"></span> discusses the impact of “ghost”
populations. He concludes that you have to be careful about which
populations you sample, but that you don’t necessarily need to sample
every population. Read the paper for the details.<a href="#fnref6"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn7"><p>As with <span><code>Migrate-N</code></span>, you can’t
calculate the likelihood explicitly, but you can approximate it
numerically. See <span class="citation"
data-cites="Nielsen-Wakeley-2001"></span> for details.<a href="#fnref7"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn8"><p>Available from <a
href="https://bio.cst.temple.edu/~hey/software/software.htm"
class="uri">https://bio.cst.temple.edu/~hey/software/software.htm</a>.<a
href="#fnref8" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn9"><p>The discrepancy was largest when divergence from the
widespread ancestor was assumed to be very recent.<a href="#fnref9"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn10"><p>All of this information is from the introduction
to <span class="citation" data-cites="Estoup-etal-2004"></span>.<a
href="#fnref10" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn11"><p>More accurately, what Peter Beerli, Joe Felsenstein,
Rasmus Nielsen, John Wakeley, and Jody Hey did.<a href="#fnref11"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn12"><p>The actual implementation is a bit more involved than
this, but that’s the basic idea.<a href="#fnref12" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn13"><p>You’re welcome to read the Methods in <span
class="citation" data-cites="Beaumont-etal-2002"></span>, and feel free
to ask questions if you’re interested. I have to confess that there’s a
decent chance I won’t be able to answer your question until I’ve done
some further studying. I’ve only used ABC a little, and I haven’t used
it for anything that I’ve publishedyet.<a href="#fnref13"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn14"><p>OK. This maybe calling it “relatively straightforward”
is misleading. Even this simplified outline is fairly complicated, but
compared to some of what you’ve already survived in this course, it may
not look too awful.<a href="#fnref14" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn15"><p>You could use any one of a variety of different
distance measures. A simple Euclidean distance might be useful, but you
could also try something more complicated, like a Mahalanobis
distance.<a href="#fnref15" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn16"><p>I know what you’re thinking to yourself now. This
doesn’t sound very simple. Trust me. It is as simple as I can make it.
The actual procedure involves local linear regression. I’m also not
telling you how to go about picking <span
class="math inline">\(\delta\)</span> or how to pick “appropriate”
summary statistics. There’s a fair amount of “art” involved in that.<a
href="#fnref16" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn17"><p>Or at least something resembling the real world<a
href="#fnref17" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn18"><p>And notice that these are 90% credible intervals,
rather than the conventional 95% credible intervals, which would be even
broader.<a href="#fnref18" class="footnote-back"
role="doc-backlink">↩︎</a></p></li>
<li id="fn19"><p>Emphasis on “relatively”.<a href="#fnref19"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</aside>
</body>
</html>
